	Although the \gls{GWAS} in \glng{scz} seems to return a lot of interesting results, the question remains: How much of the genetic variations of \glng{scz} were captured by \gls{GWAS}?
	To answer the question, we need to estimate the heritability based on the \gls{GWAS} data. 
	The challenge however, was that in order to acquire sufficient participants for the association studies, participants were randomly sampled from the population where they were usually not related to each other.
	
	\subsection{\glng{gcta}}
	As the participants were unrelated, one will need to estimate the relativeness between individuals to estimate the heritability.
	Given the genotypes of each individuals were known, one can estimate the ``genetic distance'' between two individual, which can be used to represent their relativeness.
	By calculating the ``genetic distance'' between all individuals within the \gls{GWAS}, one can obtain the \gls{grm} \citep{Yang2011}.
	
	Given that the genotypes were usually coded as 0, 1 or 2 (homozygous reference, heterozygous and homozygous alternative respectively), the genotypes follow the binomial distribution.
	The expected mean and variance of a genotype $i$ will be $2p_i$ and $2p_i(1-p_i)$ respectively where $p_i$ is the frequency of the reference allele.
	Based on this information, \citet{Yang2011} estimates the genetic relationship between individuals $j$ and $k$ as:
	\begin{equation}
	A_{jk} = \frac{1}{N}\sum^N_{i=1}\frac{(x_{ij}-2p_i)(x_{ik}-2p_i)}{2p_i(1-p_i)}
	\end{equation}
	where $x_{ij}$ is the number of copies of the reference allele for the $i^{th}$ \gls{SNP} of the $j^{th}$ individual.
	The effects of all the \glspl{SNP} were then fitted as random effects by a \gls{mlm}
	\begin{align}
	\boldsymbol{y} &= \boldsymbol{X\beta}+\boldsymbol{g}+\epsilon\\
	\mathrm{Var}(\boldsymbol{y}) &= \boldsymbol{A}\sigma_g^2+\boldsymbol{I}\sigma_\epsilon^2
	\end{align}
	where $\boldsymbol{y}$ is an $n\times 1$ vector of phenotypes with $n$ samples, $\beta$ is a vector of fixed effects such as sex and age, $\boldsymbol{g}$ is an $n\times 1$ vector of the total genetic effects of the individuals, $\sigma_g^2$ is the variance explained by all the \glspl{SNP} and finally, $\sigma_\epsilon^2$ is the variance explained by residual effects.
	
	The main concept of the method is that instead of testing the associations for individual \glspl{SNP}, one fit the effects of all SNPs as random effects in a \gls{mlm} and estimate a single parameter, i.e. the variance explained by all \glspl{SNP} or \gls{SNP}-heritability.
	Given the information of the \gls{grm}, \citet{Yang2011} implemented the \gls{reml} using the average information algorithm to estimates the $\sigma_g^2$ and $\sigma_\epsilon^2$where the \gls{reml} is a form of maximum likelihood estimation that allows unbiased estimates of variance and covariance parameters.
	The heritability of the trait is then defined as $\frac{\sigma_g^2}{\sigma_g^2+\sigma_e^2}$.
	
	Based on the above concept, \citet{Yang2010a} were able to estimate the variance in height explained by \glspl{SNP} from the height \gls{GWAS} to be around 45\%, much larger than previously reported 5\%.
	The main difference in the estimates was because the \gls{mlm} \gls{reml} were able to consider all \glspl{SNP} simultaneously without limited on significant \glspl{SNP}.
	Although the estimates was still less than 80\% which was the expected heritability of height, \citet{Yang2010a} was able to demonstrated that one possible source of ``missing heritability'' might be due to incomplete \gls{LD}.
	By taking into consideration of incomplete \gls{LD}, it was estimated that the proportion of variance explained by causal variants can be as high as 0.84 with \gls{se} of 0.16 \citep{Yang2010a}, close to the expected heritability.
	Together, \citet{Yang2011} provide a possible method for the estimation of the variance explained by \glspl{SNP} in \gls{GWAS} data and the method is now implemented in \gls{gcta} which is wildly adopted.
	
	The problem with \gls{gcta} was that genotype data are required to calculate the \gls{grm}.
	For complex disease like \glng{scz}, the data were usually obtained from multiple data source where the raw genotypes were usually unavailable due to privacy concerns.
	Instead, summary statistics were usually provided.
	Therefore estimation of variance explained by \glspl{SNP} in these \gls{GWAS} can only rely on the summary statistics. 
	  
	\subsection{\glng{ldsc}}
	An important observation of \gls{GWAS} results was that sometimes a general inflation of test statistics can be observed. 
	It was usually considered to be contributed to the presence of confounding factors such as population stratification, under the assumption that most of the \glspl{SNP} should have no association to the disease.
	It was therefore a common practice for one to perform the \gls{gc} on the \gls{GWAS} results \citep{Zheng2006}.
	
	The problem of the \gls{gc} was that the basic assumption of a small number of causal \glspl{SNP} might not be true. 
	Through careful simulation, \citet{Yang2011b} demonstrated that in the absence of population stratification and other form of technical artifacts, the presence of polygenic inheritance can also inflate the test statistic \citep{Yang2011b}.
	More importantly, they observed that the magnitude of inflation was determined by the \emph{heritability}, the \gls{LD} structure, sample size and the number of causal \glspl{SNP} of the trait.

	Following on this observation, \citet{Bulik-Sullivan2015} developed the \gls{ldsc}.
	The fundamental concept of \gls{ldsc} was that the more genetic variants a \gls{SNP} tag (e.g. in high \gls{LD} with many other \glspl{SNP}), the more likely for it to be able to tag a causal variant; 
	whereas population stratification and cryptic relatedness should not be associated with \gls{LD}. 
	The number of genetic variants tagged by a \gls{SNP}$_j$ ($l_j$)(\gls{LD} score) was then defined as the sum of $r^2$ of $k$ \glspl{SNP} within a 1cM window of \gls{SNP}$_j$:
	\begin{equation}
	l_j = \sum_kr^2_{jk}
	\label{eq:ldScore}
	\end{equation}
	
	When there is no confounding factors, the expected $\chi^2$ of \gls{SNP}$_j$ can be defined as a function of the \gls{LD} score ($l_j$), the number of samples ($N$), the number of \glspl{SNP} in the analysis($M$) and most importantly, the heritability ($h^2$):
	\begin{equation}
	\mathrm{E}[\chi^2_j | l_j] = \frac{Nh^2}{M}l_j+1
	\label{eq:fixedLDSC}
	\end{equation}
	%TODO understand the exact calculation of LDSC

	Interestingly, when confounding factors were present in the study (e.g. population stratification), \cref{eq:fixedLDSC} simply becomes
	\begin{equation}
	\mathrm{E}[\chi^2_j | l_j] = \frac{Nh^2}{M}l_j+Na+1
	\label{eq:fullLDSC}
	\end{equation}
	where $a$ is the contribution of confounding bias.
	%The equation becomes more complicated when one want to incorporate the contribution of confounding factors to the equation.
	%For example, large scale \gls{GWAS} can often suffer from population stratification.
	%To account for the population stratification, \citet{Bulik-Sullivan2015} introduced the genetic drift term $f$ as $f\sim N(0, F_{ST}V)$ where $V$ is a correlation matrix and $F_{ST}$ is Wright's Fixation index ($F_{ST}$).
	%They made the assumption where the LD Score of variant $j$ in the populations are the same.
	%Although this assumption is not reasonable when the population is very different (e.g. they are from different continents), the authors argues that they were only interested in modeling the the population stratification that might remain \emph{after} principal components analysis in \gls{GWAS} and thus their assumption are reasonable.
	%Furthermore, it was noted that there was a large correlation observed between the \gls{LD} score for all pairs of 1000 Genomes European subpopulations. 
	% I disagree with them here. It is uncertain how the correction of the population stratification affects the LD pattern or the relationship between the test statistics, thus one require a more rigid study before such a strong assumption can be made.
	
%	If one express the \gls{LD} score and the $\chi^2$ as vectors ($\boldsymbol{L}$ and $\boldsymbol{\chi^2}$) respectively, \cref{eq:fullLDSC} becomes a regression of the $\chi^2$ against the \gls{LD} score:
%	\begin{equation}
%	\boldsymbol{\chi^2}= \frac{N}{M}\boldsymbol{L}h^2+Na+1
%	\label{eq:ldReg}
%	\end{equation}
	
	As a result of that, the heritability $h^2$ will be the slope of the regression and the intercept minus one will represent the mean contribution of the confounding bias such as those of population stratification. 
	Thus, \cref{eq:fullLDSC} can be used for the estimation of heritability given that only the summary statistics and population \gls{LD} were provided. 
	
	
	Although \gls{ldsc} can be used for heritability estimation, the main focus of \citet{Bulik-Sullivan2015}'s paper was to delineating the confounding factors from the 	polygenicity of a trait.
	To test whether if \gls{ldsc} can delineate the confounding factors such as cryptic relationship and population stratification from the polygenicity of a trait, \citet{Bulik-Sullivan2015} simulated multiple \gls{GWAS} where the trait can have a polygenic architecture or where confounding factors can present.
	When the simulated trait is polygenic and no confounding factors were presented, the average \gls{ldsc} intercept was close to one, as expected by the authors.
	Only when the number of causal variants was small will the standard error of the estimates become very large.
	However, even with the enlarged variance, the estimates remained unbiased.
	On the other hand, when the \gls{GWAS} was simulated with only the confounding factors such as population stratification, the intercept estimated was approximately equal to the \gls{gc} inflation factor with only a small positive bias in the regression slope.
	
	The most important simulation was to investigate the performance of \gls{ldsc} in \gls{GWAS} of a polygenic trait where confounding factors were present. 
	It was found that even when both polygenicity and confounding factors were presented in the \gls{GWAS}, the intercept of \gls{ldsc} was approximately equal to the mean $\chi^2$ statistic among null \glspl{SNP}, providing strong evidence that \gls{ldsc} can partition the inflation in test statistic even in the presence of both bias and polygenicity.
	
	Although the main focus of \citet{Bulik-Sullivan2015} was not the estimation of \gls{SNP}-heritability in their paper, they did provide an estimate of variance explained by the \glspl{SNP} in the \gls{pgc} \glng{scz} \gls{GWAS} \citep{Ripke2014}.
	By applying the liability threshold adjustment, \citet{Bulik-Sullivan2015} estimated the \gls{SNP}-heritability of \glng{scz} should be 0.555 with \gls{se} of 0.008.
	The estimated \gls{SNP} heritability was lower than the heritability estimated from population based study (64\% \citep{Lichtenstein2009}) and twin studies (81\% \citep{Sullivan2003}) suggesting that it is possible for variants other than common \glspl{SNP} were accounting for the variation in \glng{scz}.
	An example will be \gls{cnv} which was also found to be associated with \glng{scz} \citep{Szatkiewicz2014}.
	
	Another possibility of the``missing'' heritability can be due to interaction between the genetic and environmental factors. 
	Although previous studies \citep{Gottesman01071967} suggested that the non-additive genetic factors were unlikely to contribute to \glng{scz}, the possibility of involvement of gene-environmental interaction $G\times E$ were not ruled out.
	Indeed, in the adoption study conducted by \citet{Tienari2004}, it was found that individuals with higher genetic risk were significantly more sensitive to ``adverse'' vs ``healthy'' rearing patterns in adoptive families than are adoptees at low genetic risk \citep{Tienari2004}, providing support to a possible interaction between genetic and environmental factors.
	Therefore, in order to account for the ``missing'' heritability, one might need to consider genetic variations other than \glspl{SNP} and might need to take into consideration of the $G\times E$ interaction.
	On the other hand, as demonstrated by \citet{Yang2010a}, the ``missing'' heritability might simply because of incomplete \gls{LD} between the \glspl{SNP} and the causal variants. 
	A possible method would be to perform sequencing studies variants across the whole genome can be detected at the same time.
	
	Nonetheless, the \gls{SNP}-heritability estimation from \citet{Ripke2014} were still encouraging, as for the first time in genetic research of \glng{scz}, a large portion of heritability of \glng{scz} were finally identified.
	This permit the genetic research of \glng{scz} to move beyond statistical association and focus on the functional basis of the genetic susceptibility locus of \glng{scz}.
	
	\subsection{Partitioning of Heritability of Schizophrenia}
	\subsectionmark{Partitioning of Heritability}
	Traditionally, functional enrichment analysis in \gls{GWAS} only take into account of \glspl{SNP} that passed the genome wide significance threshold. 
	However, for complex traits such as that of \glng{scz}, much of the heritability might lies in \glspl{SNP} that do not reach genome wide significance threshold at the current sample size.
	For example, in 2013, only 13 risk loci were detected using 13,833 \glng{scz} samples and 18,310 controls \citep{Ripke2013}. 
	When the sample size increased to 34,241 \glng{scz} samples and 45,604 controls in 2014, 108 risk loci were identified \citep{Ripke2014}. 
	Thus, if one only consider the significant loci, risk loci that have not reach genome wide significance threshold might be ignored from the analysis, decreasing the power of the functional enrichment analysis.
	
	Unlike traditional functional enrichment analysis, \gls{ldsc} uses information from all \glspl{SNP} and taking into account of the \gls{LD} structure to partition heritability into different functional categories. 
	It should therefore be more powerful when compared to traditional analysis and should help to provide useful insight into the disease etiology of \glng{scz}.

	\citet{Finucane2015} used data from \citet{Ripke2014} and functional categories derived from the ENCODE annotation \citep{ENCODEProjectConsortium2012}, the NIH Roadmap Epigenomics Mapping Consortium annotation \citep{Bernstein2010} and other studies \citep{Finucane2015} to identify functional categories that were most enriched in \glng{scz}.
	In their study, it was found that brain cell types were most enriched in \glng{scz}, especially those related to the \gls{cns}.
	Of all the functional categories, the most enriched category in \glng{scz} was the H3K4me3 mark in the fetal brain(\cref{tab:cellTypeScz}). 
	As H3K4me3 was mostly linked to active promoters, this suggest that genes that were activated in fetal brain (e.g. genes related to brain development) were associated with \glng{scz}, supporting the idea of \glng{scz} as a neuro-developmental disorder. 
	
	Moreover, it was also observed that the second most enriched cell types were those related to immunity.
	Undoubtedly, the \gls{cns} and the immune system have an important role in the disease etiology of \glng{scz}. 

	\begin{singlespace}
	\begin{longtable}{p{6cm}rrr}
		%\begin{tabular}{rrrr}
			\toprule
			Cell type & cell-type group & Mark  & P-value \\
			\midrule
			Fetal brain** & CNS   & H3K4me3 & $3.09\times 10^{-19}$ \\
			Mid frontal lobe** & CNS   & H3K4me3 & $3.63\times 10^{-15}$ \\
			Germinal matrix** & CNS   & H3K4me3 & $2.09\times 10^{-13}$ \\
			Mid frontal lobe** & CNS   & H3K9ac & $5.37\times 10^{-12}$ \\
			Angular gyrus** & CNS   & H3K4me3 & $1.29\times 10^{-11}$ \\
			Inferior temporal lobe** & CNS   & H3K4me3 & $1.70\times 10^{-11}$ \\
			Cingulate gyrus** & CNS   & H3K9ac & $5.37\times 10^{-11}$ \\
			Fetal brain** & CNS   & H3K9ac & $5.75\times 10^{-11}$ \\
			Anterior caudate** & CNS   & H3K4me3 & $2.19\times 10^{-10}$ \\
			Cingulate gyrus** & CNS   & H3K4me3 & $4.57\times 10^{-10}$ \\
			Pancreatic islets** & Adrenal/Pancreas & H3K4me3 & $2.24\times 10^{-09}$ \\
			Anterior caudate** & CNS   & H3K9ac & $3.16\times 10^{-9}$ \\
			Angular gyrus** & CNS   & H3K9ac & $4.68\times 10^{-9}$ \\
			Mid frontal lobe** & CNS   & H3K27ac & $7.94\times 10^{-9}$ \\
			Anterior caudate** & CNS   & H3K4me1 & $1.20\times 10^{-8}$ \\
			Inferior temporal lobe** & CNS   & H3K4me1 & $3.72\times 10^{-8}$ \\
			Psoas muscle** & Skeletal Muscle & H3K4me3 & $4.17\times 10^{-8}$ \\
			Fetal brain** & CNS   & H3K4me1 & $6.17\times 10^{-8}$ \\
			Inferior temporal lobe** & CNS   & H3K9ac & $9.33\times 10^{-8}$ \\
			Hippocampus middle** & CNS   & H3K9ac & $9.33\times 10^{-7}$ \\
			Pancreatic islets** & Adrenal/Pancreas & H3K9ac & $1.62\times 10^{-6}$ \\
			Penis foreskin melanocyte primary** & Other & H3K4me3 & $2.09\times 10^{-6}$ \\
			Angular gyrus** & CNS   & H3K27ac & $2.34\times 10^{-6}$ \\
			Cingulate gyrus** & CNS   & H3K4me1 & $2.82\times 10^{-6}$ \\
			Hippocampus middle** & CNS   & H3K4me3 & $2.82\times 10^{-6}$ \\
			CD34 primary** & Immune & H3K4me3 & $4.68\times 10^{-6}$ \\
			Sigmoid colon** & GI    & H3K4me3 & $5.01\times 10^{-6}$ \\
			Fetal adrenal** & Adrenal/Pancreas & H3K4me3 & $6.31\times 10^{-6}$ \\
			Inferior temporal lobe** & CNS   & H3K27ac & $8.32\times 10^{-6}$ \\
			Peripheralblood mononuclear primary** & Immune & H3K4me3 & $9.33\times 10^{-6}$ \\
			Gastric** & GI    & H3K4me3 & $1.17\times 10^{-5}$ \\
			Substantia nigra* & CNS   & H3K4me3 & $1.95\times 10^{-5}$ \\
			Fetal brain* & CNS   & H3K4me3 & $2.63\times 10^{-5}$ \\
			Hippocampus middle* & CNS   & H3K4me1 & $3.31\times 10^{-5}$ \\
			Ovary* & Other & H3K4me3 & $6.46\times 10^{-5}$ \\
			CD19 primary (UW)* & Immune & H3K4me3 & $7.08\times 10^{-5}$ \\
			Small intestine* & GI    & H3K4me3 & $8.51\times 10^{-5}$ \\
			Lung* & Cardiovascular & H3K4me3 & $1.17\times 10^{-4}$ \\
			Fetal stomach* & GI    & H3K4me3 & $1.29\times 10^{-4}$ \\
			Fetal leg muscle* & Skeletal Muscle & H3K4me3 & $1.51\times 10^{-4}$ \\
			Spleen* & Immune & H3K4me3 & $1.70\times 10^{-4}$ \\
			Breast fibroblast primary* & Connective/Bone & H3K4me3 & $2.04\times 10^{-4}$ \\
			Right ventricle* & Cardiovascular & H3K4me3 & $2.14\times 10^{-4}$ \\
			CD4+ CD25- Th primary* & Immune & H3K4me3 & $2.19\times 10^{-4}$ \\
			CD4+ CD25- IL17- PMA Ionomycin stim MACS Th sprimary* & Immune & H3K4me1 & $2.19\times 10^{-4}$ \\
			CD8 naive primary (UCSF-UBC)* & Immune & H3K4me3 & $2.24\times 10^{-4}$ \\
			Pancreas* & Adrenal/Pancreas & H3K4me3 & $2.34\times 10^{-4}$ \\
			CD4+ CD25- Th primary* & Immune & H3K4me1 & $2.75\times 10^{-4}$ \\
			CD4+ CD25- CD45RA+ naive primary* & Immune & H3K4me1 & $2.75\times 10^{-4}$\\
			Colonic mucosa* & GI    & H3K4me3 & $3.24\times 10^{-4}$ \\
			Right atrium* & Cardiovascular & H3K4me3 & $3.31\times 10^{-4}$ \\
			Fetal trunk muscle* & Skeletal Muscle & H3K4me3 & $3.39\times 10^{-4}$ \\
			CD4+ CD25int CD127+ Tmem primary* & Immune & H3K4me3 & $3.47\times 10^{-4}$ \\
			Substantia nigra* & CNS   & H3K9ac & $3.63\times 10^{-4}$ \\
			Placenta amnion* & Other & H3K4me3 & $4.17\times 10^{-4}$ \\
			Breast myoepithelial* & Other & H3K9ac & $5.50\times 10^{-4}$ \\
			CD8 naive primary (BI)* & Immune & H3K4me1 & $5.75\times 10^{-4}$ \\
			Substantia nigra* & CNS   & H3K4me1 & $6.61\times 10^{-4}$ \\
			Cingulate gyrus* & CNS   & H3K27ac & $7.94\times 10^{-4}$ \\
			CD4+ CD25- CD45RA+ naive primary* & Immune & H3K4me3 & $8.71\times 10^{-4}$ \\
			\bottomrule
		%\end{tabular}%
		\caption[Enrichment of Top Cell Type of Schizophrenia]{Enrichment of Top Cell type of Schizophrenia.
			* = significant at False Discovery Rate $<$ 0.05.
			** = significant at p $<$ 0.05 after correcting for multiple hypothesis. 
			Reproduce with permission from Journal.\citep{Finucane2015}}
		\label{tab:cellTypeScz}%
	\end{longtable}%
	\end{singlespace}
	
%	\subsection{Genetic Correlation}
%	Another very important application of \gls{ldsc} is that it allow one to identify the genetic correlation between traits\citep{Bulik-Sullivan2015a}. 
%	The genetic correlation can be used as a genetic analogue to co-morbidity, thus allowing deeper understanding to the etiology of the traits.
%	Above all, genetic correlation was important in studying the treatment response. 
%	It has been observed that there was an increased prevalence of anxiety, depression and substance abuse in \glng{scz} \citep{Buckley2009}. 
%	These co-morbidity were generally associated with more severe psychopathology and with poorer outcome \citep{Buckley2009}.
%	A deeper understanding of possible co-morbidity between different traits and \glng{scz} might provide insight not only to the disease etiology of \glng{scz}, it might even provide important information in possible treatment options for \glng{scz}. 
%	Using breast cancer as an example, it was found that patients with comorbidity had poorer survival than those without comorbidity \citep{Sogaard2013} and it was suggested that by treating the comorbid diseases, one might be able to delay mortality in breast cancer patients \citep{Ording2013}.
%		
%	By applying their method to 25 different phenotypes, \citet{Bulik-Sullivan2015a} shown that \glng{scz} has significant genetic correlation with bipolar disorder, major depression and more surprisingly, anorexia nervosa.
%	Previous studies have always suggest there to be a co-morbidity between \glng{scz} and bipolar disorder \citep{Lichtenstein2009,Purcell2009,Buckley2009}.
%	Similarly, it was not uncommon for \glng{scz} to display depressive symptoms \citep{Buckley2009}. 
%	It was even observed that individuals at high risk and ultrahigh risk for developing \glng{scz} have generally demonstrated a significant degree of depressive symptoms prior to and during the emergence of psychotic symptoms, suggesting a close relationship between \glng{scz} and depression. 
%	
%	On the other hand, the genetic correlation between \glng{scz} and anorexia nervosa were slightly unexpected for there has been a lack of study in the co-morbidity between eating disorder and \glng{scz}. 
%	Nonetheless, this finding raises the possibility of similarity between anorexia and nervosa.
	% Serotonergic system have been implicated in depression, negative symptoms of sczhiophrneia and eating disorders \citep{Arranz2007}.