\documentclass[12pt]{scrbook}
\usepackage{amsmath}
\usepackage{bm}
\usepackage{colortbl}
\usepackage{graphicx}
\usepackage{caption}
\usepackage{fullpage}
\usepackage{afterpage}
\usepackage{float}
\usepackage{multirow}
\usepackage[nodisplayskipstretch]{setspace}
\usepackage{booktabs}
\usepackage{textcomp}
\usepackage{gensymb}
\usepackage[utf8]{inputenc} 
%\usepackage{parskip}
\usepackage{xr}
\usepackage{pdflscape}
\usepackage[labelfont=bf,tableposition=top]{caption}
\usepackage[inline]{enumitem}
\usepackage{wrapfig}
\usepackage{longtable}
\usepackage{subfig}
\usepackage[hidelinks]{hyperref}
\usepackage{geometry}
\usepackage{fancyhdr}
\usepackage[acronym,nomain,toc,makeindex ]{glossaries}
\usepackage{graphicx}
\usepackage[section]{placeins}
\usepackage{eufrak}
\usepackage[table]{xcolor} 
\usepackage{cmap}
\usepackage{cleveref} %This need to be the last package for it to work properly 
\usepackage[hyperref=true,maxcitenames=3,maxbibnames=3,url=false,doi=false,isbn=false,backref=true,natbib=true,backend=biber,sorting=nyvt,defernumbers=false, style=authoryear]{biblatex}
\setcounter{secnumdepth}{3}
%
%
%	TITLE PAGE DEFINITION
%
%
\renewcommand*\contentsname{Contents}

\newcommand{\rom}[1]{\uppercase\expandafter{\romannumeral #1\relax}}
\newcommand*{\glng}{\glsentrylong}
\newcommand*{\Glng}{\Glsentrylong}
\newcommand{\beginsupplement}{%
	\setcounter{table}{0}
	\renewcommand{\thetable}{S\arabic{table}}%
	\setcounter{figure}{0}
	\renewcommand{\thefigure}{S\arabic{figure}}%
}
\newcommand{\specialcell}[2][c]{\begin{tabular}[#1]{@{}c@{}}#2\end{tabular}}

%
%	GLOSSARY SECTION
%
\addbibresource[datatype=bibtex]{citation/citation.bib}
\makeglossary
\newacronym{scz}{SCZ}{schizophrenia}
\newacronym[longplural={Genome Wide Association Studies}]{GWAS}{GWAS}{Genome Wide Association Study}
\newacronym{wgs}{WGS}{Whole Genome Sequencing}
\newacronym{SNP}{SNP}{Single Nucleotide Polymorphism}
\newacronym{LD}{LD}{Linkage Disequilibrium}
\newacronym{PGS}{PGS}{Polygenic Risk Score}
\newacronym{tSVD}{tSVD}{Truncated Singular Value Decomposition}
\newacronym{SVD}{SVD}{Singular Value Decomposition}
\newacronym{shrek}{SHREK}{SNP HeRitability Estimation Kit}
\newacronym{gcta}{GCTA}{Genome-wide Complex Trait Analysis}
\newacronym{ldsc}{LDSC}{LD SCore regression}
\newacronym{CEU}{CEU}{Northern Europeans from Utah}
\newacronym{se}{SE}{standard error}
\newacronym{maf}{MAF}{Minor Allele Frequency}
\newacronym{isc}{ISC}{International Schizophrenia Consortium}
\newacronym{mgs}{MGS}{Molecular Genetic of Schizophrenia}
\newacronym{sgene}{SGENE}{Schizophrenia Genetics Consortium}
\newacronym{pgc}{PGC}{Psychiatric Genomics Consortium}
\newacronym{rin}{RIN}{RNA integrity number}
\newacronym{gd}{GD}{Gestation Day}
\newacronym{rpkm}{RPKM}{Reads Per Kilobase per Million mapped reads}
\newacronym{wgcna}{WGCNA}{Weighted Gene Co-expression Network Analysis}
\newacronym{pc}{PC}{Principle Component}
\newacronym{GO}{GO}{Gene Ontology}
\newacronym{MAGMA}{MAGMA}{Multi-marker Analysis of GenoMic Annotation}
\newacronym{ngs}{NGS}{next generation sequencing}
\newacronym{dsm}{DSM}{Diagnostic and Statistical Manual of Mental Disorders}
\newacronym{mse}{MSE}{mean squared error}
\newacronym{who}{WHO}{World Health Organization}
\newacronym{yld}{YLD}{years lost due to disability}
\newacronym{ci}{CI}{confidence interval}
\newacronym{iq}{IQ}{Intelligence Quotient}
\newacronym{polyic}{PolyI:C}{polyriboinosinic-polyribocytidilic acid}
\newacronym{lps}{LPS}{lipopolysaccharide}
\newacronym{il6}{IL-6}{Interleukin-6}
\newacronym{mz}{MZ}{monozygotic}
\newacronym{dz}{DZ}{dizygotic}
\newacronym{mia}{MIA}{maternal immune activation}
\newacronym{ncp}{NCP}{non-centrality parameter}
\newacronym{cnv}{CNV}{copy number variation}
\newacronym{grm}{GRM}{Genetic Relationship Matrix}
\newacronym{gc}{GC}{Genomic Control}
\newacronym{mhc}{MHC}{major histocompatibility complex}
\newacronym{cns}{CNS}{central nervous system}
\newacronym{panss}{PANSS}{Positive and Negative Symptom Scale}
\newacronym{fga}{FGA}{First Generation Antipsychotic}
\newacronym{sga}{SGA}{Second Generation Antipsychotic}
\newacronym{nmda}{NMDA}{N-methyl-D-aspartate}
\newacronym{eps}{EPS}{extrapyramidal motor symptoms}
\newacronym{td}{TD}{tardive dyskinesia}
\newacronym{pet}{PET}{positron emission tomography}
\newacronym{catie}{CATIE}{Clinical Antipsychotic Trials of Intervention Effectiveness}
\newacronym{pca}{PCA}{principle component analysis}
\newacronym{fdr}{FDR}{false discovery rate}
\newacronym{pcgc}{PCGC}{Phenotype correlation - genotype correlation regression}
\newacronym{scidp}{SCID-P}{Structured Clinical Interview for DSM-IV-TR Axis I Disorders, Research Version, Patient Edition}
\newacronym{ibd}{IBD}{identity by descent}
\newacronym{bp}{bp}{base pair}
\newacronym{mb}{mb}{megabase}
\newacronym{kb}{kb}{kilobase}
\newacronym{deg}{DEG}{differentially expressed gene}
\newacronym{qc}{QC}{quality control}
\newacronym{hku}{HKU}{the University of Hong Kong}
\newacronym{pcr}{PCR}{polymerase chain reaction}
\newacronym{ercc}{ERCC}{External RNA Controls Consortium}
\newacronym{go}{GO}{Gene Ontology}
\newacronym{rtpcr}{rt-PCR}{real time PCR}
\newacronym{ct}{CT}{cycle threshold}
\newacronym{geo}{GEO}{Gene Expression Omnibus}
\newacronym{mlm}{MLM}{mixed linear model}
\newacronym{reml}{REML}{restricted maximum likelihood}
\newacronym{pufa}{PUFA}{polyunsaturated fatty acid}
\newacronym{lrt}{LRT}{likelihood ratio test}
\newacronym{msigdb}{MSigDB}{Molecular Signatures Database}
\newacronym{fgf}{FGF}{fibroblast growth factor}
\newacronym{egf}{EGF}{epidermal growth factor}
\newacronym{pi3k}{PI3K}{phosphatidylinositol 3-kinase}
\newacronym{mapk}{MAPK}{mitogen-activated protein kinase}
\newacronym{gaba}{GABA}{$\gamma$-aminobutyric acid}
\newacronym{ecm}{ECM}{extracellular matrix}
\newacronym{mmp}{MMP}{matrix metalloproteinase}
\newacronym{qqplot}{QQ-plot}{quantile-quantile Plot}
\newacronym{cm}{cM}{centiMorgan}
\newacronym{mrna}{mRNA}{messenger RNA}
\newacronym{psd}{PSD}{postsynaptic density}
\newacronym{kegg}{KEGG}{Kyoto Encyclopedia of Genes and Genomes}
\newacronym{arc}{ARC}{neuronal activity-regulated cytoskeleton-associated protein}
\newacronym{cb1}{CB1}{cannabinoid receptor type 1}
%
%
%	FORMATING SECTION
%
%
\pagestyle{fancy}
\fancyhf{}
\fancyfoot[LE,RO]{\thepage}
\renewcommand{\footrulewidth}{1pt}
\fancyhead[LE]{\leftmark}
\fancyhead[RO]{\rightmark}

\geometry{
	top=1in,            % <-- you want to adjust this
	inner=1in,
	outer=3.5cm,
	bottom=3.5cm,
	headheight=3ex,       % <-- and this
	headsep=2ex,          % <-- and this
}


\raggedbottom %Remove it before printing as this is something to do with global settings. Can make each page look uneven but more dense. 
%\onehalfspacing
\doublespacing
%TODO this is the include space
%\includeonly{heritability_estimation}
%\includeonly{omega_project}
%\includeonly{heritability_estimation,omega_project}
%\includeonly{literature_review}Z


%\extrafloats{100}






\makeindex
\begin{document}\thispagestyle{empty}
\pagestyle{empty}

%\maketitle
\begin{singlespace}
	\begin{titlepage}
		\begin{center}
			\vspace*{1cm}
			
			\Huge
			\textbf{The Genetics of Schizophrenia - The Contribution of Common Variants and Genes Implicated in a Maternal Immune Activation Model}
			
			\vspace{0.5cm}
			\LARGE
			
			\vspace{1.5cm}
			
			\textbf{\href{mailto:choishingwan@gmail.com}{Choi Shing Wan}}
			
			\vfill
			
			A thesis submitted in partial fulfillment of the requirements for \\
			the Degree of Doctor of Philosophy
			
			\vspace{0.8cm}
			
			\includegraphics[width=0.4\textwidth]{figure/hkuLogo.jpg}
			
			\Large
			Department of Psychiatry\\
			University of Hong Kong\\
			Hong Kong\\
			\today
		\end{center}
	\end{titlepage}
\end{singlespace}


\frontmatter 
	\cleardoublepage
	\phantomsection
	\addcontentsline{toc}{chapter}{Abstract}
	\chapter*{\centerline{Abstract}}
	\vspace{0.1cm}
	\Glng{scz} is a disabling disorder affecting approximately 1\% of the population worldwide.
	To fully understand disease mechanisms for the development of effective treatments, it is important not only to examine how certain genetic polymorphisms can predispose individuals to develop, but also how environmental factors triggers onset of the disorder in apparently healthy individuals. 
	
	\gls{GWAS} is now a standard approach for investigating associations of common genetic variations (mainly \glspl{SNP}) with \glng{scz}. 
	A recent meta-analysis of \gls{GWAS} of \glng{scz} has identified 108 loci significantly associated with \glng{scz}. 
	However, it is possible for other genetic factors such as rare variants to contribute to the disease etiology of \glng{scz}. 
	Estimating the overall contribution of common \glspl{SNP} to \glng{scz} therefore has important implications for future research strategy.
	
	In this thesis, we propose a novel approach for estimating the contribution of \glspl{SNP} to \glng{scz} (\gls{SNP}-heritability) from \gls{GWAS} summary statistics, called the \gls{shrek}.
	Our simulation results suggest that, when compared to an existing method, \gls{ldsc}, \gls{shrek} provided a more robust estimate for oligo-gentic traits and in case-control designs where no confounding variables was present. 
	Using the summary statistics from the latest meta-analysis of \gls{GWAS} on \glng{scz}, we estimated that \glng{scz} has a \gls{SNP}-heritability of 0.185 (SD=0.00450), which is similar to the estimate of 0.198 (SD=0.0057) by \gls{ldsc}.
	The result indicate that common \glspl{SNP} have a relatively small contribution to the genetic predisposition to \glng{scz} when compared to the overall heritability estimated from twin and family studies. 
	Therefore, alternative strategies like whole-genome sequencing may be necessary for identifying additional \glng{scz} genes.
	
	Previous studies have reported the interaction between genetic variation and prenatal infection in the etiology of \glng{scz}.
	There are evidences that the effect of prenatal infection is mediated by maternal immune response, thus it is likely for the perturbation induced by \gls{mia} to interact with genetic variations in the development of \glng{scz}.
	
	We therefore performed a RNA-sequencing study to investigate whether there are any genetic overlaps between differential genes induced by \gls{mia} and genetic variations detected by \glng{scz} \gls{GWAS} using the \gls{polyic} mouse model.
	We found that the functional gene sets associated with \glng{scz} are also enriched in \gls{mia}.
	In addition, when investigating the treatment effect of n-3 \gls{pufa} rich diet in \gls{mia}, we found that the gene expression of \textit{Sgk1}, a gene that regulates the glutamatergic system, is affected by the n-3 \gls{pufa} rich diet in the \gls{polyic} exposed mice. 
	It is therefore possible for \textit{Sgk1} to mediate the treatment effect of n-3 \gls{pufa} rich diet in the \gls{mia} model. 
	In conclusion, our results suggested that genes related to neural function and calcium ion signaling, as well as glutamate-related genes such as \textit{Sgk1}, are the potential targets for future \glng{scz} research.
	
	\begin{flushright}
		(467 words)
	\end{flushright}
	\glsresetall
	\cleardoublepage
	\phantomsection
	\addcontentsline{toc}{chapter}{Declaration}
	\newgeometry{inner=2in, outer=2in}
	\chapter*{\centerline{Declaration}}
	\vspace{1cm}
	I declare that this thesis represents my own work, except where due acknowledgments	is made, and that it has not been previously included in a thesis, dissertation or report submitted to this University or to any other institution for a degree, diploma or other qualification.
	\vspace{1.5cm}
	
	\centerline{Signed....................................................................}
	\centerline{Choi Shing Wan}
	\restoregeometry
	\cleardoublepage
	\phantomsection
	\addcontentsline{toc}{chapter}{Acknowledgments}
	\chapter*{\centerline{Acknowledgements}}
	\vspace{1cm}
	I would like to express my deepest gratitude to Professor Pak Sham.
	I am eternally grateful for his trust, supervision, patience and support in the course of my study.
	I would also like to thanks Dr Stacey Cherny and Dr Wanling Yeung for giving me valuable advice for my projects.
	My special thanks go to Dr Johnny Kwan.
	He has provided critical advices on my projects and has taught me a great deal in the field of statistic.
	
	The past 4 years has been a blast and I really enjoy my time in this department.
	This is only possible because of all the great people here.
	Thank you Beatrice Wu, Dr Li Qi, Tomy Hui, Vicki Lin, Nick Lin, John Wong, Dr Clara Tang, Dr Amy Butler, Dr Emily Wong, Dr Allen Gui, Dr Sylvia Lam, Yung Tse Choi, Oi Chi Chan, Pui King Wong and Dr Miaoxin Li, without you everything will be much different.
	I will forever cherish the time I spent with you. 
	
	Words alone cannot express my gratitude to Beatrice Wu and my family.
	Their support and encouragement have been my greatest source of energy and have helped me to continue on with my study. 
	
	\begin{center}
	\large THANK YOU!
	\end{center}
	\cleardoublepage
	\phantomsection
	\begin{singlespace}
	\printglossary[title=Abbreviations,toctitle=Abbreviations,style=long,nonumberlist]
	\cleardoublepage
	\phantomsection
	%To generate the correct abbreviations, use alt+shift+F1 twice before using F1
	
	\cleardoublepage
	\phantomsection
	\addcontentsline{toc}{chapter}{Contents}
	
		\setcounter{tocdepth}{3}
		\tableofcontents
		\listoffigures
		\listoftables
	\end{singlespace}
\mainmatter
\pagestyle{fancy}

\setlength{\parindent}{4em}
\setlength{\parskip}{0.75em}
	
	\include{literature_review}
	\include{heritability_estimation}
	\include{omega_project}
	\chapter{Conclusion}
	\label{conclusionChapter}
	\glsresetall
	It has been a long time since \glng{scz} was recognized as a genetic disorder. 
	After years of failure in identifying robust genetic markers associated with \glng{scz}, the \Glng{scz} Working group of \gls{pgc} has recently identified 108 independent genetic loci associated with \glng{scz} \citep{Ripke2014}.
	
	The positive results from the \gls{pgc} \glng{scz} \gls{GWAS} is highly encouraging, yet unclear whether additional resources should be invested into collecting more \gls{GWAS} samples, or whether other genetic approaches such as exome or whole genome sequencing should be performed to examine the role of rare variants. 
	Therefore, estimating the true contribution of all the \gls{GWAS} \glspl{SNP} to the genetic predisposition of individuals to \glng{scz} (i.e. \gls{SNP} heritability) has important implications for future research strategy.
	% Know whether to increase sample size or just plain right do something else
	
	Problem with the estimation of \gls{SNP} heritability is that the sample genotypes from \gls{pgc} \glng{scz} \gls{GWAS} is unavailable, therefore the \gls{grm} cannot be computed. 
	Methods such as \gls{gcta} \citep{Yang2011} and \gls{pcgc} \citep{Golan2014}, which relies on the \gls{grm} to estimate the \gls{SNP} heritability are therefore not infeasible.
	Recently, \gls{ldsc} \citep{Bulik-Sullivan2015}, which utilize the summary statistics from \gls{GWAS} instead of \gls{grm}, was developed for the estimation of \gls{SNP} heritability.
	Therefore, it is now possible to estimate the \gls{SNP} heritability of \glng{scz}.
	
	Nevertheless, it is important to assess the performance of \gls{ldsc} before applying it to the actual \glng{scz} data.
	Herein, we performed extensive simulations to investigate the impact of different genetic architectures and sampling strategies on the performance of \gls{ldsc}.
	As \citet{Bulik-Sullivan2015} reported that \gls{ldsc} under-perform in certain scenarios, we also developed \gls{shrek}, an alternative algorithm that uses summary statistics from \gls{GWAS} to estimate the \gls{SNP} heritability of a trait. 
	
	Our simulation results suggest that \gls{shrek} provided a more robust estimate for oligogentic traits and in case-control designs in which no confounding variables was present.
	However, it is also observed that estimates from \gls{ldsc} with fixed intercept and \gls{shrek} are inflated in case control designs.
	The magnitude of inflation is proportional to the population prevalence of the trait, where lower population prevalence results in a larger upward bias. 
	Therefore, further development is required in order to obtain an accurate estimation of \gls{SNP} heritability in case-control settings.
	
	Using summary statistics from \gls{pgc} \glng{scz} \gls{GWAS}, we estimated that \glng{scz} has a \gls{SNP} heritability of 0.185 (SD=0.00450), which is similar to the estimate of 0.198 (SD=0.0057) by \gls{ldsc} when intercept estimation was not performed.
	When intercept estimation was perform, \gls{ldsc} estimated the \gls{SNP} heritability of \glng{scz} to be around 0.135 (SD=0.00720), which is lower than the other estimates. 
	This result indicated that common \glspl{SNP} have relatively less contribution to the genetic predisposition of individuals to \glng{scz} as measured by the heritability estimated
	Therefore, alternative strategies like whole genome sequencing, which allows for the identification of rare mutations and structural variation,  would be more efficient for identifying additional susceptibility loci associated with \glng{scz}, compared to \gls{GWAS}.
	
	Another possible source of the ``missing'' heritability is the interaction between genetic and environmental factors ($G\times E$). 
	Previous studies have reported the interaction between genetic variations and prenatal infection \citep{Tienari2004,Clarke2009}. 
	As the effect of prenatal infection was mainly mediated by maternal immune response \citep{Brown2010}, it is likely for the perturbation induced by \gls{mia} to interacts with genetic variations in the development of \glng{scz}.
	
	To investigate whether the perturbation induced by \gls{mia} also affect the same functional gene sets as genetic variations associated with \glng{scz}, we performed an RNA sequencing on the \gls{polyic} \gls{mia} mouse model. 
	As converging evidence from \gls{GWAS}, \gls{cnv} and sequencing studies suggest rare and common variants in genes that related to \gls{psd} \citep{Purcell2014,Consortium2015a} and calcium ion channels \citep{Purcell2014,Ripke2014,Szatkiewicz2014} are contributing to the etiology of \glng{scz}, these gene sets were selected for the gene set enrichment analysis. 
	
	Our results suggest that with the exception of the \gls{psd} gene set obtained from \citet{Purcell2014}, all gene sets related to \gls{psd} and calcium ion channel signalling are found to be significantly perturbed in \gls{mia} samples. 
	Furthermore, the gene set containing genes within the associated \gls{GWAS} LD-intervals from \citet{Purcell2014} is also found to be significant.
	This strongly suggest that the perturbation induced by early \gls{mia} events are acting on the same functional gene sets as the genetic variation associated with \glng{scz}.
	\gls{psd} and calcium ion channel are therefore important target for subsequent research in \glng{scz}.
	
	On the other hand, a number of studies have reported the potential of n-3 \gls{pufa} in the treatment of \glng{scz} \citep{Li2015,Trebble2003}.
	Therefore, we also investigated the effect of n-3 \gls{pufa} rich diet in samples exposed to early \gls{mia} events.
	The analysis allowed us to identify \textit{Sgk1}, a gene that regulates the glutamatergic system, to be significantly differentiated in \gls{polyic} exposed mouse receiving different diet. 
	In addition, the \gls{psd} gene set obtained form the \gls{go} is also found to be significant in \gls{polyic} exposed mouse receiving different diet. 
	We therefore speculate that the n-3 \gls{pufa} rich diet might indirectly enhance the expression of \textit{Sgk1} or increase the expression of the \gls{psd} proteins in \gls{polyic} exposed mice, which may compensating for the reduced neural functioning, therefore reduces the \glng{scz}-like behaviours.
	
	Finally, we estimated the relative contribution of the gene sets to the \gls{SNP} heritability of \glng{scz}. 
	However, only the \gls{psd} gene set from \gls{go}, which was previous reported to be enriched by common variants \citep{Consortium2015a}, and the gene set containing genes within the associated \gls{GWAS} LD-intervals from \citet{Purcell2014} are found to contribute a disproportionate amount of \gls{SNP} heritability in \glng{scz}.
	The relative contribution of the gene sets are only around 1\% of the \gls{SNP} heritability, suggesting that other functional gene sets might be accounting for the remaining \gls{SNP} heritability.
	
	It might be worthwhile to repeat the analysis in \citet{Consortium2015a} using \gls{ldsc}.
	The problem of the partitioning of heritability analysis is that the enrichment can easily be driven by a small number of genes with large effect size. 
	Therefore, further developments might be required in order to prevent the spurious enrichment. 
	
	\section{Schizophrenia: Future Perspectives}
	We are now entering a new era of sequencing, where a wide variety of tools are at our disposal to identify different genetic variations associating with a disease. 
	It is foresee that the ability to investigate the whole genome/exome at a per base resolution will allow for the identification of more genetic variations such as the rare mutations, that are associated with \glng{scz}.
	
	With the sophistication of technologies, whole genome sequencing can now be performed with the HiSeq \RN{10} Ten system, costing less than \$1,000, allowing for more data to be generated. 
	While the technologiy advancement have allowed a massive increase in available sequencing data, the bioinformatic analysis are now becoming the bottleneck of genetic research.
	For example, the alignment of sequence read to low complexity sequence or low-degeneracy repeats remains challenging and error prone, having a negative impact to the quality of the results \citep{Sims2014}. 
	New sequencing technology such as Oxford Nanopore, which can provide extra long-reads, might help to make alignment easier by providing extra information for each individual reads.
	However, the Oxford Nanopore is still under development and has a relatively high error rate \citep{Mikheyev2014}. 
	Only until the error rate is dramatically decreased can the use of Oxford Nanopore system become feasible. 
		
	Even if the reads can be perfectly aligned to the genome, the functional annotation of variants remains challenging.
	For complex disease such as \glng{scz}, a large amount of susceptibility loci can be observed throughout the genome, yet estimation of the functional impact can only be performed on variants located within the exomic regions.
	The development of ENCODE project \citep{ENCODEProjectConsortium2012} and Genotype-Tissue Expression (GTEx) project \citep{Consortium2015} have helped to provide reference point for the annotation of genetic variations in the intergenic regions.
	However, the functional impact of many genetic variation in the genome remains unknown. 
	Only through the tireless effort of the molecular biologist can sufficient information be gained to allow for the detail annotation of the data.

	Moreover, epigenetic studies in \glng{scz} \citep{Wockner2014,Nishioka2012} have identified genes with differential DNA methylation patterns associated with \glng{scz}, whereas etiology studies have reported the possibility of interaction between prenatal infection and genetic variation in risk of developing \glng{scz} \citep{Tienari2004,Clarke2009}, suggesting the possible involvement of epigenetics and $G\times E$ interaction in the etiology of \glng{scz}.
	Therefore, it is important to combine information of multiple dimensions -- including genetic, \gls{cnv}, epigenetic changes, genetic and environmental interaction, expression, structural properties and spatial organization of chromosomes --  can we understand the etiology of \glng{scz}.
		% GxE 
		% Prenatal infection -> only rodant information
		% Need planning (birth cohort)
		
		% Future prospective
		% More functional studies
		% Increase sample size
		% Epigenetic studies

	
%	\begin{figure}
%		\centering
%		\includegraphics[width=0.7\textwidth]{figure/maf_effectSize.png}
%		\caption[Relationship between Effect Size and Allele Frequency]{
%			Relationship between effect size and allele frequency. 
%			It is expected that rare variants with large effect size were actively selected against in the population and therefore should be rare.
%		}
%		\label{fig:effectSize}
%	\end{figure}
	
%	Currently, most of the focus in \glng{scz} was directed to genetic variation yet it is possible that the heritability of \glng{scz} is also transmitted in the form of epigenetic changes such as methylation.
%	It was observed that the risk for individual born from a schizophrenic mother is larger than that from a schizophrenic father. 
%	This suggests that maternal specific elements, such as maternal imprinting and mitochondria might account for part of the risk of \glng{scz}. 
%	Epigenetic studies in \glng{scz} \citep{Wockner2014,Nishioka2012} has identified genes with differential DNA methylation patterns associated with \glng{scz}, suggesting the importance of epigenetics in the etiology of \glng{scz}.
%	
%	As a highly heritable disorder, most of the research of \glng{scz} has been focusing on the genetic factors. 
%	Although the genetic variation accounted for majority of the variations in \glng{scz}, the environmental factors, especially prenatal infection is also an important factor to consider. 
%	It was estimated that prenatl infection accounts for roughly 33\% of all \glng{scz} cases \citep{Brown2010}.
%	The \gls{mia} rodent model has provide vital information on the possible interaction between the immune and neuronal system in the etiology of \glng{scz} \citep{Meyer2007a}.
%	For example, \gls{il6}, a pro-inflammatory cytokine has been found to be an important mediator in generating the schizophrenia-like behaviour in rodent model \citep{Smith2007}.
%	More importantly, there are evidence of the interaction between prenatal infection and genetic variation, supporting a mechanism of gene-environment interaction in the causation of \glng{scz} \citep{Clarke2009}.
%	As the \gls{SNP}-heritability estimation does not take into account of the gene environmental interactions, it is possible that the ``missing'' heritability can be due to gene-environmental interactions. 
%	Efforts is now made by the European network of national \glng{scz} networks studying Gene-Environmental Interaction (EUGEI) to identify possible genetic and environmental interaction that contributes to the disease etiology of \glng{scz}.
%	
%	
%	
%	% This is only the beginning, not the end
%	In conclusion, we have only catch a glimpse of the etiology of \glng{scz} and there are still a lot of questions left unanswered.
%	It is expected that only by combining the study of epigenetic, genomic variation, gene expressions, and gene environmental interaction can provide a deeper understanding of the complex disease mechanism of \glng{scz} be obtained.
%	
	
	
	\backmatter
	\printbibliography[heading=bibintoc,title={Bibliography}]

	%\input{environmental_risk/er_rtPCR_primer.tex}
\end{document}


