\documentclass{book}
\usepackage{amsmath}
\usepackage{bm}
\usepackage{colortbl}
\usepackage{graphicx}
\usepackage{caption}
\usepackage{fullpage}
\usepackage{afterpage}
\usepackage{multirow}
\usepackage{setspace}
\usepackage{booktabs}
\usepackage{gensymb}
\usepackage{parskip}
\usepackage{xr}
\usepackage{pdflscape}
\usepackage[inline]{enumitem}
\usepackage{wrapfig}
\usepackage{longtable}
\usepackage{subfig}
\usepackage[natbib=true, style=numeric-comp,subentry,backend=biber,sorting=none]{biblatex}
\addbibresource{citation/Thesis.bib}
\usepackage[hidelinks]{hyperref}
\usepackage{fancyhdr}
\usepackage{fixltx2e}
\usepackage[acronym,nomain,toc,makeindex ]{glossaries}



\pagestyle{fancy}
\fancyhead[LE,RO]{\itshape \nouppercase \rightmark}
\fancyhead[LO,RE]{\itshape \nouppercase Chapter \arabic{chapter}}
\setlength{\headheight}{40.0pt}
\setlength{\headsep}{0.2in}
\addtolength{\topmargin}{-4\baselineskip}
\renewcommand{\headrulewidth}{0.4pt}
\renewcommand{\footrulewidth}{0.4pt}
\newcommand{\beginsupplement}{%
	\setcounter{table}{0}
	\renewcommand{\thetable}{S\arabic{table}}%
	\setcounter{figure}{0}
	\renewcommand{\thefigure}{S\arabic{figure}}%
}

\title{Genetic and Environmental risk factors of Schizophrenia and Autism}
\date{\today}
\author{\href{mailto:choishingwan@gmail.com}{Choi Shing Wan}\\
	\includegraphics[width=0.5\textwidth]{hkuLogo.jpg}}
\singlespacing
\renewcommand*\contentsname{Contents}
\onehalfspacing
%\doublespacing


%\includeonly{environmental_risk/er_chapter,supplementary_materials}

\makeglossary
\newacronym{SCZ}{SCZ}{Schizophrenia}
\newacronym{ASD}{ASD}{Autism Spectrum Disorder}
\newacronym[longplural={Genome Wide Association Studies}]{GWAS}{GWAS}{Genome Wide Association Study}
\newacronym{SNP}{SNP}{Single Nucleotide Polymorphism}

\makeindex
\begin{document}\thispagestyle{empty}
\pagestyle{empty}

%\maketitle
\begin{titlepage}
	\begin{center}
		\vspace*{1cm}
		
		\Huge
		\textbf{Heritability Estimation and Risk Prediction in Schizophrenia}
		
		\vspace{0.5cm}
		\LARGE
		
		\vspace{1.5cm}
		
		\textbf{\href{mailto:choishingwan@gmail.com}{Choi Shing Wan}}
		
		\vfill
		
		A thesis submitted in partial fulfillment of the requirements for \\
		the Degree of Doctor of Philosophy
		
		\vspace{0.8cm}
		
		\includegraphics[width=0.4\textwidth]{hkuLogo.jpg}
		
		\Large
		Department of Psychiatry\\
		University of Hong Kong\\
		Hong Kong\\
		\today
		
	\end{center}
\end{titlepage}


\frontmatter 

	\cleardoublepage
	\phantomsection
	\addcontentsline{toc}{chapter}{Declaration}
	\chapter*{Declaration}
	\cleardoublepage
	\phantomsection
	\addcontentsline{toc}{chapter}{Acknowledgments}
	\chapter*{Acknowledgements}
	\cleardoublepage
	\phantomsection
	\printglossary[title=Abbreviations,toctitle=Abbreviations]
	\cleardoublepage
	\phantomsection
	%To generate the correct abbreviations, use alt+shift+F1 twice before using F1
	
	\cleardoublepage
	\phantomsection
	\addcontentsline{toc}{chapter}{Contents}
	\begin{singlespace}
		\tableofcontents
	\end{singlespace}
\mainmatter
	\chapter*{Introduction}
		\addcontentsline{toc}{chapter}{Introduction}
	\chapter*{Some considerations}
	\begin{enumerate}
		\item PRSice requires the phenotype to aid its selection (More information= stronger)
		\item It seems like LDSC doesn't necessary perform badly in oligogenic situation.
		Rather, it is that when the trait is oligogenic, it is more likely for LDSC to behaviour in a strange way.
		\item For each condition: extreme phenotype, quantitative trait, case control, we can have a separated review. 
		Discuss on the benefits and challenges of each condition and the method we deal with them.
		So we can have two chapters (case control, quantitative trait) where extreme phenotype can be a big subsection within quantitative trait.
		\item For each chapter, there will be this introduction (review on the method), our methodology (Calculation, implementation and also simulation), result (the simulation result). 
		Then we can have the application (PGC, network)
	\end{enumerate}
	
	\chapter{Literature Review}
	\section{Twin Studies}
	Should briefly talk about how Twin modeling was used for finding the GE contribution.
	Should also mention the ACE model.
	At the end, we can talk about the heritability estimates of SCZ and AD
	\section{Searching for Genetic Variants}
	\subsection{Role of Common Variants}
	\subsubsection{Genome Wide Association Study}
	Should talk about what is GWAS and how it is used.
	Should also talk about the current GWAS studies in SCZ and AD
	\subsection{Role of Rare Variants}
	\subsubsection{Exome Sequencing}
	Similar to the GWAS.
	Talk about the Pros and Cons.
	Need to briefly mention the Denovo paper and Shaun's paper.					
	\subsubsection{Whole Genome Sequencing}
	Very very brief description of WGS and the current status.
	\section{Narrow Sense Heritability}
	\section{Risk Prediction}
	\section{Summary}
	
	\chapter{Heritability Estimation}
	This chapter should be used in similar way as the general method section in Clara's thesis. Considering that the subsequent chapters all rely on this implementation.
	\section{Introduction}
	\section{Methodology}
		The narrow-sense heritability is defined as 
		$$
			h^2 = \frac{var(X)}{var(Y)}
		$$
		where $var(X)$ is the variance of the genotype and $var(Y)$ is the variance of the phenotype.
		In a \acrfull{GWAS}, regression were performed between the \acrshort{SNP}s and the phenotypes, giving
		\begin{equation}
			Y=\beta X+\epsilon
			\label{standardRegress}
		\end{equation}
		where $Y$ and $X$ are the standardized phenotype and genotype respectively. 
		$\epsilon$ is then the error term, accounting for the non-genetic elements contributing to the phenotype (e.g. Environment factors).
		Based on equation \ref{standardRegress}, one can then have
		\begin{align}
			var(Y) = var(\beta X)+ var(\epsilon) \nonumber\\
			var(Y) = \beta^2var(X) \nonumber\\
			\beta^2\frac{var(X)}{var(Y)}= 1
			\label{betaHeri}
		\end{align}
		$\beta^2$ is then considered as the portion of phenotype variance explained by the variance of genotype. 
		Which can also be considered as the narrow-sense heritability of the phenotype.
		
		It is noted that as both $X$ and $Y$ are standardized, $\beta^2$ will be equal to the coefficient of determination ($r^2$). 
		
		A challenge in calculating the heritability from \acrshort{GWAS} data is that usually only the test-statistic or p-value were provided and one will not be able to directly calculate the heritability based on equation \ref{betaHeri}. In order to estimation the heritability of a trait from the \acrshort{GWAS} test-statistic, we rely on the properties of the Pearson product-moment correlation coefficient:
		\begin{equation}
			r = \frac{t}{\sqrt{n-2+t^2}}
			\label{pearsonProduct}
		\end{equation}
		where $t$ follows the student-t distribution and $n$ is the number of samples.
		One can then obtain the $r^2$ by taking the square of \ref{pearsonProduct}
		\begin{equation}
			r^2 = \frac{t^2}{n-2+t^2}
			\label{oriRSquared}
		\end{equation}
		It is observed that $t^2$ will follow the F-distribution and when $n$ is big, $t^2$ will converge into $\chi^2$ distribution.
		
		As under the null distribution, $t^2$ should have mean approximately equal to 1, we then define the \acrshort{SNP} contribution ($f$) as:
		\begin{equation}
		f = \frac{t^2-1}{n-2+t^2}
		\label{snpContribution}
		\end{equation} 
		
		When all the \acrshort{SNP}s were independent, the heritability of the phenotype can be simply defined as
		\begin{equation}
			h^2 = \sum^m_1{f}
			\label{simpleHeritability}
		\end{equation}
		where $m$ is the number of \acrshort{SNP}.
		
		Considering that one of the main concept in \acrshort{GWAS} is to be be able to ``tag'' the true causal variants using common \acrshort{SNP}s based on the linkage disequilibrium between the \acrshort{SNP}, it is impractical to assume the \acrshort{SNP}s to be independent from each other. 
		When linkage disequilibrium exists between the \acrshort{SNP}s, equation \ref{simpleHeritability} will provide an over-estimation of the heritability. 
		In order to obtain an unbiased estimation of the heritability of the phenotype, one must take into account of the linkage structure between the \acrshort{SNP}s.
	\subsection{Quantitative Trait}
	\subsection{Case Control Studies}
	\subsection{Extreme Phenotype Selections}
	\section{Simulation}
	\subsection{Quantitative Trait}
	\subsection{Case Control Studies}
	\subsection{Exreme Phenotype Selections}
	\section{Result}
	\section{Discussion}
	
	\chapter{Heritability of Schizophrenia}
	\section{Introduction}
	\section{Heritability Estimation}
	This will be a very simple section, focused on how to perform the heritability estimation on \acrfull{SCZ}.
	Should also tokenize the heritability into subcategories (e.g. immune, neuron, etc)
	\subsection{Methodology}
	\subsection{Result}
	\section{Brain development and Schizophrenia}
	Here we will perform the WGCNA and brain development network.
	Seeing how the whether if any brain development network were enriched with SNPs that explain the variance of phenotype
	\subsection{Methodology}
	\subsection{Result}
	\section{Discussion}
	\chapter{Heritability of Response to antipsychotic treatment}
	\section{Introduction}
	Here we try to use Beatrice's data and estimate the heritability explained in drug response.
	Should also repeat the region-wise heritability
	\section{Methodology}
	\section{Result}
	\section{Discussion}
	\chapter{Risk Prediction}
	\section{Methodology}
	\subsection{Simulation}
	\section{Result}
	\section{Discussion}
	\chapter{Conclusion}
	\backmatter
	\printbibliography
	\chapter*{Appendix}
\end{document}


