\documentclass[12pt]{book}
\usepackage{amsmath}
\usepackage{bm}
\usepackage{colortbl}
\usepackage{graphicx}
\usepackage{caption}
\usepackage{fullpage}
\usepackage{afterpage}
\usepackage{float}
\usepackage{multirow}
\usepackage[nodisplayskipstretch]{setspace}
\usepackage{booktabs}
\usepackage{textcomp}
\usepackage{gensymb}
\usepackage[utf8]{inputenc} 
%\usepackage{parskip}
\usepackage{xr}
\usepackage{pdflscape}
\usepackage[labelfont=bf,tableposition=top]{caption}
\usepackage[inline]{enumitem}
\usepackage{wrapfig}
\usepackage{longtable}
\usepackage{subfig}
\usepackage[hidelinks]{hyperref}
\usepackage{geometry}
\usepackage{fancyhdr}
\usepackage[acronym,nomain,toc,makeindex ]{glossaries}
\usepackage{graphicx}
\usepackage{cleveref} %This need to be the last package for it to work properly 
\usepackage[hyperref=true,maxcitenames=3,maxbibnames=3,url=false,isbn=false,backref=true,natbib=true,backend=biber,sorting=nyvt,defernumbers=false, style=authoryear]{biblatex}


%
%
%	TITLE PAGE DEFINITION
%
%
\title{Genetic and Environmental risk factors of Schizophrenia and Autism}
\date{\today}
\author{\href{mailto:choishingwan@gmail.com}{Choi Shing Wan}\\
	\includegraphics[width=0.5\textwidth]{hkuLogo.jpg}}
\singlespacing
\renewcommand*\contentsname{Contents}


%\includeonly{environmental_risk/er_chapter,supplementary_materials}
\newcommand{\rom}[1]{\uppercase\expandafter{\romannumeral #1\relax}}
\newcommand*{\glng}{\glsentrylong}
\newcommand*{\Glng}{\Glsentrylong}
\newcommand{\beginsupplement}{%
	\setcounter{table}{0}
	\renewcommand{\thetable}{S\arabic{table}}%
	\setcounter{figure}{0}
	\renewcommand{\thefigure}{S\arabic{figure}}%
}
\newcommand{\specialcell}[2][c]{\begin{tabular}[#1]{@{}c@{}}#2\end{tabular}}

%
%	GLOSSARY SECTION
%
\addbibresource[datatype=bibtex]{citation/citation.bib}
\makeglossary
\newacronym{scz}{SCZ}{schizophrenia}
\newacronym[longplural={Genome Wide Association Studies}]{GWAS}{GWAS}{Genome Wide Association Study}
\newacronym{wgs}{WGS}{Whole Genome Sequencing}
\newacronym{SNP}{SNP}{Single Nucleotide Polymorphism}
\newacronym{LD}{LD}{Linkage Disequilibrium}
\newacronym{PGS}{PGS}{Polygenic Risk Score}
\newacronym{tSVD}{tSVD}{Truncated Singular Value Decomposition}
\newacronym{SVD}{SVD}{Singular Value Decomposition}
\newacronym{shrek}{SHREK}{SNP Heritability and Risk Estimation Kit}
\newacronym{gcta}{GCTA}{Genome-wide Complex Trait Analysis}
\newacronym{ldsc}{LDSC}{LD SCore}
\newacronym{CEU}{CEU}{Northern Europeans from Utah}
\newacronym{se}{SE}{standard error}
\newacronym{maf}{maf}{Minor Allele Frequency}
\newacronym{isc}{ISC}{International Schizophrenia Consortium}
\newacronym{mgs}{MGS}{Molecular Genetic of Schizophrenia}
\newacronym{sgene}{SGENE}{Schizophrenia Genetics Consortium}
\newacronym{pgc}{PGC}{Psychiatric Genomics Consortium}
\newacronym{rin}{RIN}{RNA integrity number}
\newacronym{gd}{GD}{Gestation Day}
\newacronym{rpkm}{RPKM}{Reads Per Kilobase per Million mapped reads}
\newacronym{wgcna}{WGCNA}{Weighted Gene Co-expression Network Analysis}
\newacronym{pc}{PC}{Principle Component}
\newacronym{GO}{GO}{Gene Ontology}
\newacronym{MAGMA}{MAGMA}{Multi-marker Analysis of GenoMic Annotation}
\newacronym{ngs}{NGS}{next generation sequencing}
\newacronym{dsm}{DSM}{Diagnostic and Statistical Manual of Mental Disorders}
\newacronym{mse}{MSE}{mean squared error}
\newacronym{who}{WHO}{World Health Organization}
\newacronym{yld}{YLD}{years lost due to disability}
\newacronym{ci}{CI}{confidence interval}
\newacronym{iq}{IQ}{intelligence quotient}
\newacronym{polyic}{PolyI:C}{polyriboinosinic-polyribocytidilic acid}
\newacronym{lps}{LPS}{lipopolysaccharide}
\newacronym{il6}{IL-6}{Interleukin-6}
\newacronym{mz}{MZ}{monozygotic}
\newacronym{dz}{DZ}{dizygotic}
\newacronym{mia}{MIA}{maternal immune activation}
\newacronym{ncp}{NCP}{non-centrality parameter}
\newacronym{cnv}{CNV}{copy number variation}
\newacronym{grm}{GRM}{Genetic Relationship Matrix}
\newacronym{gc}{GC}{Genomic Control}
\newacronym{mhc}{MHC}{major histocompatibility complex}
\newacronym{cns}{CNS}{central nervous system}
%
%
%	FORMATING SECTION
%
%

\pagestyle{fancy}
\fancyhf{}
\fancyfoot[LE,RO]{\thepage}
\renewcommand{\footrulewidth}{1pt}
\fancyhead[LE]{\leftmark}
\fancyhead[RO]{\rightmark}

\geometry{
	top=1in,            % <-- you want to adjust this
	inner=1in,
	outer=1in,
	bottom=1in,
	headheight=3ex,       % <-- and this
	headsep=2ex,          % <-- and this
}


\raggedbottom %Remove it before printing as this is something to do with global settings. Can make each page look uneven but more dense. 
\onehalfspacing
%\doublespacing

\makeindex
\begin{document}\thispagestyle{empty}
\pagestyle{empty}

%\maketitle
\begin{titlepage}
	\begin{center}
		\vspace*{1cm}
		
		\Huge
		\textbf{Heritability Estimation and Risk Prediction in Schizophrenia}
		
		\vspace{0.5cm}
		\LARGE
		
		\vspace{1.5cm}
		
		\textbf{\href{mailto:choishingwan@gmail.com}{Choi Shing Wan}}
		
		\vfill
		
		A thesis submitted in partial fulfillment of the requirements for \\
		the Degree of Doctor of Philosophy
		
		\vspace{0.8cm}
		
		\includegraphics[width=0.4\textwidth]{figure/hkuLogo.jpg}
		
		\Large
		Department of Psychiatry\\
		University of Hong Kong\\
		Hong Kong\\
		\today
		
	\end{center}
\end{titlepage}


\frontmatter 

	\cleardoublepage
	\phantomsection
	\addcontentsline{toc}{chapter}{Declaration}
	\newgeometry{inner=2in, outer=2in}
	\chapter*{\centerline{Declaration}}
	\vspace{1cm}
	I declare that this thesis represents my own work, except where due acknowledgments	is made, and that it has not been previously included in a thesis, dissertation or report submitted to this University or to any other institution for a degree,	diploma or other qualification.
	\vspace{1.5cm}
	
	\centerline{Signed....................................................................}
	\restoregeometry
	\cleardoublepage
	\phantomsection
	\addcontentsline{toc}{chapter}{Acknowledgments}
	\chapter*{\centerline{Acknowledgements}}
	\cleardoublepage
	\phantomsection
	\begin{singlespace}
	\printglossary[title=Abbreviations,toctitle=Abbreviations]
	\cleardoublepage
	\phantomsection
	%To generate the correct abbreviations, use alt+shift+F1 twice before using F1
	
	\cleardoublepage
	\phantomsection
	\addcontentsline{toc}{chapter}{Contents}
		\tableofcontents
		\listoffigures
		\listoftables
	\end{singlespace}
\mainmatter
\pagestyle{fancy}

\setlength{\parindent}{4em}
\setlength{\parskip}{0.75em}
	%\chapter*{Some considerations}
	%\begin{enumerate}
	%	\item PRSice requires the phenotype to aid its selection (More information= stronger)
	%	\item It seems like LDSC doesn't necessary perform badly in oligogenic situation.
	%	Rather, it is that when the trait is oligogenic, it is more likely for LDSC to behaviour in a strange way.
	%	\item For each condition: extreme phenotype, quantitative trait, case control, we can have a separated review. 
	%	Discuss on the benefits and challenges of each condition and the method we deal with them.
	%	So we can have two chapters (case control, quantitative trait) where extreme phenotype can be a big subsection within quantitative trait.
	%	\item For each chapter, there will be this introduction (review on the method), our methodology (Calculation, implementation and also simulation), result (the simulation result). 
	%	Then we can have the application (PGC, network)
	%\end{enumerate}
	%Things that I have to include
	%\begin{enumerate}
	%	\item Schizophrenia
	%	\begin{enumerate}
	%		\item Case Control (PGC) \\ LDSC has basically did everything related to partitioning and genetic correlation, so we should focus on the brain network instead
	%		\item Drug Response \\ No one has done it before, should have a brief section here. 
	%		Focus should be with the \emph{heritability} of response, not to identify the variants.
	%		However, we can try to partition the heritability of drug response too.\\
	%		Is it really related to genetics? 
	%		Or is it something related to environmental?
	%		e.g. Discouraging family members leads to not adhere to medicine
	%	\end{enumerate}
	%	\item Heritability Estimation \\ This part should be straightforward, just the algorithm and the simulation results
		
	%\end{enumerate}
	
	\chapter{Introduction - Heritability Estimation in Schizophrenia}
	\chaptermark{Introduction}
	% Disease background
	\section{Schizophrenia}
	\Glng{scz} is a detrimental psychiatric disorder, affecting around $0.3\sim0.7\%$ of the population\citep{AmericanPsychiatricAssociation2013}.
	It is characterized by positive symptoms including delusions, hallucinations, disorganized speech and grossly disorganized behavior, and negative symptoms such as the diminished emotional expression\citep{AmericanPsychiatricAssociation2013} with a typical age of onset at late adolescent or late 20s in male and late 20s or early 30s in female\citep{Schultz2007}.
	
	\Glng{scz} not only impose long lasting health, social and financial burden not only to the patients, but also to their families\citep{Knapp2004}. 
	Even more so, patients with \glng{scz} increased suicide rate \citep{Saha2007}, leading to a higher mortality.
	Based on the \gls{who} report, \glng{scz} is one of the top 20 leading cause of \gls{yld} in 2012, ranking 16 among all possible causes (\cref{tab:whoYLD}), demonstrating the extent of impact from \glng{scz} to patients.
	\begin{table}[ht]
		\centering
		\caption[Top 20 leading cause of \glng{yld}]{Top 20 leading cause of \gls{yld} calculated by \gls{who} in year 2012.
			\Glng{scz} was considered as one of the top 20 leading cause of \gls{yld}\citep{Geneva2013}}
		\begin{tabular}{rrrrr}
			\toprule
			Rank  & Cause & \gls{yld} (000s) & \% \gls{yld} & \specialcell[b]{\gls{yld} per \\100k population}\\
			\midrule
			0     & All Causes & 740,545 & 100   & 10466 \\
			1     & Unipolar depressive disorders & 76,419 & 10.3  & 1080 \\
			2     & Back and neck pain & 53,855 & 7.3   & 761 \\
			3     & Iron-deficiency anaemia & 43,615 & 5.9   & 616 \\
			4     & Chronic obstructive pulmonary disease & 30,749 & 4.2   & 435 \\
			5     & Alcohol use disorders & 27,905 & 3.8   & 394 \\
			6     & Anxiety disorders & 27,549 & 3.7   & 389 \\
			7     & Diabetes mellitus & 22,492 & 3     & 318 \\
			8     & Other hearing loss & 22,076 & 3     & 312 \\
			9     & Falls & 20,409 & 2.8   & 288 \\
			10    & Migraine & 18,538 & 2.5   & 262 \\
			11    & Osteoarthritis & 18,096 & 2.4   & 256 \\
			12    & Skin diseases & 15,744 & 2.1   & 223 \\
			13    & Asthma & 14,134 & 1.9   & 200 \\
			14    & Road injury & 13,902 & 1.9   & 196 \\
			15    & Refractive errors & 13,498 & 1.8   & 191 \\
			16    & Schizophrenia & 13,408 & 1.8   & 189 \\
			17    & Bipolar disorder & 13,271 & 1.8   & 188 \\
			18    & Drug use disorders & 10,620 & 1.4   & 150 \\
			19    & Endocrine, blood, immune disorders & 10,495 & 1.4   & 148 \\
			20    & Gynecological diseases & 10,227 & 1.4   & 145 \\
			\bottomrule
		\end{tabular}%
		\label{tab:whoYLD}%
	\end{table}%
	
	Due to the severity of \glng{scz}, it has drawn much attention from the research community aiming to delineate the disease mechanics and be able to identify the risk factors.
	Arguably, the most important first step to any \glng{scz} study is to have a robust and reliable disease diagnosis.
	
	\section{Diagnosis}
	% Describe early problems of diagnosis
	% Describe current method of diagnosis
	\Glng{scz} was first named ``Dementia Praecox'' by Dr. Emil Kraepelin and was later renamed as \glng{scz} by Dr. Eugen Bleuler\citep{Jablensky2010}.
	Early nosological entity for \glng{scz} such as that in \gls{dsm}-\rom{1} and \gls{dsm}-\rom{2} were vague and unreliable where the inter-rater agreement can be as low as 54$\%$.\citep{Tsuang2000,Harvey2012} 
	
	Later nosologies addressed these problem by introducing structural assessment and clear defined criteria. 
	With these improvements, the inter-rater agreement of \gls{dsm}-\rom{3} raised to $\sim 90\%$ \citep{Harvey2012}, suggesting the diagnosis were much more reliable.
	
	Currently \gls{dsm} is at its 5th edition\citep{AmericanPsychiatricAssociation2013}. 
	A patient will be diagnosed with \glng{scz}(F20.9) if they suffered from 2 or more of the following symptoms for a significant portion of time during a 1-month period: 
	\begin{enumerate*}[label=\arabic*\upshape)]
		\item delusion; \label{ls:delusion}
		\item hallucinations;\label{ls:hallucinations}
		\item disorganized speech;\label{ls:disorganizedSpeech}
		\item grossly disorganized or catatonic behaviour; and\label{ls:catatonicBehavior}
		\item negative symptoms such as diminished emotional expression,\label{ls:negativeSymptoms}
	\end{enumerate*}  where one of the symptom must be either (\ref{ls:delusion}, (\ref{ls:hallucinations} or (\ref{ls:disorganizedSpeech}.
	Signs of disturbance also need to persist for at least 6-month before the patient can be diagnosed with \glng{scz}.
	
	\section{Risk Factors of Schizophrenia}
	\sectionmark{Risk Factors}
	% Talk about brown's study, the increased risk of schizophrenia in the influenza epidemic
	% Talk about some other risk factors such as drug use etc
	Considerable effort has been made trying ot identify possible risk factors of \glng{scz}. 
	It was first observed that there was an increased risk of \glng{scz} in individual who were fetuses during the 1957 influenza epidemic\citep{Mednick1958}. 
	Subsequently, other infectious agents such as HSV-2 and \textit{T.gondii} were also found to increase the risk of \glng{scz} if an individual's mother were infected during pregnancy.
	As different infectious agents all increase the risk of \glng{scz}, it leads to the hypothesis of \gls{mia} \citep{Brown2010}.
	It was hypothesized that instead of a particular infectious agents, it was the maternal immune response that disrupt the brain development in the offspring, thus leading to an elevated risk of \glng{scz}.
		
	By utilizing the rodent models, it was found when the pregnant rodent was injected with the viral mimic \gls{polyic} or the bacterial \gls{lps}, the offspring will display neuropathological features similar to those observed in \glng{scz}\citep{Meyer2009b}.
	It was further demonstrated that similar findings can be obtained through the injection of only the \gls{il6}\citep{Smith2007}, suggesting that it was not the infection, but the maternal immune response that might have disrupted the fetal brain development.
	
	% If have time, can extend here
	
	Recent studies of global gene expression patterns in \gls{mia}-exposed rodent fetal brains \citep{Garbett2012a} suggest that \gls{mia} only causes a transient gene expression change in the fetal brain.
	Based on their observation, the author suggest that it was likely for the post-pubertal onset of schizophrenic and other psychosis-related phenotypes to be stemmed from attempts of the brain to counteract the environmental stress induced by \gls{mia} during its early development\citep{Garbett2012a}. 
	
	\begin{figure}
		\centering
		\includegraphics[width=\textwidth]{figure/risk_factors_of_schizophrenia.png}
		\caption[Risk factors of \glng{scz}]{Risk factors of \glng{scz}.
			It was observed that family history of \glng{scz} was the largest risk factors.
			Risk of \glng{scz} can be more than 9 times higher than the general population for individual with a family history of \glng{scz}}
		\label{fig:riskfactors}
	\end{figure}
	Together, these results supports the involvement of \gls{mia} in the development of \glng{scz}.
	It was even estimated that one third of all \glng{scz} cases could have been prevented shall all infection were prevented from the entire pregnant population\citep{Brown2010}.
	
	Similarly, tobacco consumption \citep{Kelly1999}, socio economical status and even the area of birth (e.g. urban vs suburb) were also found to be associated with increased risk of schizophrenia\citep{McGrath2008a}.
	However, by and large, the single largest risk factor was family history of \glng{scz}(\cref{fig:riskfactors})\citep{Sullivan2005}.
	Studies conducted by Ernst R{\"u}din, Franz J. Kallmann and Hans Luxenburger, all demonstrated that the relatives of \glng{scz} tends to have increased risk of \glng{scz}\citep{Gottesman1982}. 
	The implication of such observation was two fold:
	as family members usually shares larger portion of their genetic effects with each other than that of the population, the genetic effects might be the main mediator of \glng{scz}; 
	on the other hand, culture, socio-economical status and area of birth usually also transmit within the family, so one cannot separate the environmental factors from the genetic factors.


	It was important to study the relative contribution of genetic and environmental influence to individual differences in \glng{scz}.
	If \glng{scz} was indeed a genetic disease, we may then focus the resources into study of genetic variations in \glng{scz} patients. 
	To quantify the relative contribution of genetic and environmental influence, one will need to estimate the \emph{heritability} of \glng{scz}.
	
	\section{Broad Sense Heritability}

	A key concept in quantitative genetics is \emph{heritability}, which was defined as \emph{proportion} of total variance of a trait in a population explained by variation of genetic factors in the population.
	One can partition observed phenotype into a combination of genetic and environmental components\citep{Falconer1996}
	$$
	\text{Phenotype(P)}=\text{Genotype(G)}+\text{Environment(E)}
	$$
	where the variance of the observed phenotype ($\sigma_P^2$) can be expressed as variance of genotype ($\sigma_G^2$) and variance of environment ($\sigma_E^2$)
	$$
		\sigma_P^2=\sigma_G^2+\sigma_E^2
	$$
	The broad sense heritability can then be defined as the ratio between the variance of the observed phenotype and the variance of the genetic effects
	$$
	H^2=\frac{\sigma_G^2}{\sigma_P^2}
	$$
	
	One key feature of heritability is that it is a \emph{ratio} of \emph{populational} measurement at a specific time point.
	As a result of that, the heritability estimation might differ from one population to another due to difference in \gls{maf} and one might obtain a different heritability estimate if the method or time-point of measurement of the trait differs because of different environmental factors coming into play.
	An classical example was the study of \gls{iq} where the heritability estimation increases with age\citep{Bouchard2013}.
	It was hypothesize that the shared environment has a larger effect on individuals when they were young, and that as they become more independent, the effect of shared environment diminishes, leading to a \emph{increased portion} of variance in \gls{iq} explained by the variance in genetic\citep{Bouchard2013}. 
	
	\section{Narrow sense Heritability}
	In reality, the problem of heritability was more complicated for there were different forms of genetic effects. 
	For example, one can partition the genetic variance into variance of additive genetic effects ($\sigma_A^2$), variance of dominant genetic effects ($\sigma_D^2$) and other epistatic genetic effects ($\sigma_I^2$) such that
	$$
		\sigma_G^2=\sigma_A^2+\sigma_D^2+\sigma_I^2
	$$
	where additive genetic variance was the variance explained by the average effects of all loci involved in the determination of the trait, whereas dominant genetic effects and epistatic genetic effectss were the interaction between alleles at the \emph{same} locus or \emph{different} loci respectively.
	
	As individuals only transmit one copy of each allele to their offspring, relatives other than full siblings and identical twins will only share a maximum of one copy of the allele from each other.
	Considering that dominance and non-additive genetic effects were concerning the interactive effect, which usually involve more than one copy of the alleles, these effects are unlikely to contribute to the resemblance between relatives \citep{Visscher2008}.
	On the other hand, the additive genetic effects is usually transmitted from parent to offspring, thus it is usually more useful to consider the narrow sense heritability($h^2$) which only consider the additive genetic effects:
	\begin{align}
	h^2&=\frac{\sigma_A^2}{\sigma_P^2} \notag\\
	h^2&=\frac{\sigma_A^2}{\sigma_G^2+\sigma_E^2}
	\label{eq:narrowHeritability}
	\end{align}
	
	% add in how to calculate the heritability in normal way here
	To obtain the additive genetic effect, we can first consider the genetic effect of parents to be $G_p=A+D$. 
	As only half of the additive effect were transmitted to their offspring, the child will have a genetic effect of $G_c=\frac{1}{2}A+\frac{1}{2}A'+D'$ where $A'$ is the additive genetic effect obtained from another parent by random and $D'$ is the non-additive genetic effect in the offspring.
	If we then consider the parent offspring covariance, we will get
	\begin{align}
	\mathrm{Cov_{OP}}&= \sum(\frac{1}{2}A+\frac{1}{2}A'+D')(A+D)\notag\\
	&=\frac{1}{2}\sum A^2+\frac{1}{2}\sum AD + \frac{1}{2}\sum A'(A+D) +D'(A+D) \notag\\ 
	&=\frac{1}{2}V_A+ \frac{1}{2}\mathrm{Cov}_{AD} + \frac{1}{2}\mathrm{Cov}_{A'A} + \frac{1}{2}\mathrm{Cov}_{A'D} +\mathrm{Cov}_{D'A} +\mathrm{Cov}_{D'D}  
	\label{eq:halfCompletedCovOP}
	\end{align} 
	Under the assumption of random mating,  $A'$ should be independent from $A$ and $D$. 
	On the other hand, as $D'$ was specific to the child, both of them should be independent from $A$ and $D$.
	Moreover, the covariance between the additive genetics and non-additive genetics should be zero\citep{Falconer1996}.
	Thus, \cref{eq:halfCompletedCovOP} becomes
	\begin{align}
	\mathrm{Cov_{OP}} &= \frac{1}{2}V_A+\mathrm{Cov}_{AD} \notag\\
	&= \frac{1}{2}V_A
	\label{eq:covOP}
	\end{align}
	Now if we assume the variance of phenotype of the parent and offspring were the same, then using \cref{eq:covOP}, we can obtain the narrow-sense heritability as
	\begin{align}
	h^2 &= \frac{1}{2}\frac{V_A}{\sigma_P^2}
	\label{eq:narrowHerit}
	\end{align}
	If we consider the simple linear regression equation $Y=X\beta+\epsilon$, its slope can be calculated as 
	\begin{equation}
	\beta_{XY} = \frac{\mathrm{Cov}_{XY}}{\sigma_{X}{Y}}
	\end{equation}
	which resemble \cref{eq:narrowHerit}. 
	Therefore,  we can calculate the narrow sense heritability as
	\begin{equation}
	h^2 = 2\beta_{OP}
	\label{eq:narrowSenseHerit}
	\end{equation}
	where $\beta_{OP}$ is the slope of the simple linear regression regressing the phenotype of an offspring to the phenotype of \emph{one} of its parents.
	We can further generalize \cref{eq:narrowSenseHerit} to all possible relativeness 
	\begin{equation}
	h^2=\frac{\beta_{XY}}{r}
	\label{eq:finalNarrow}
	\end{equation}
	where $r$ is the relativeness of $X$ and $Y$.
	
	A key assumption in this calculation was that the relatives does not share anything other than the additive genetic factors.
	However, this was usually not the case as relatives does tends to be in the same cultural group and might have similar socio-economical status which might all contribute the the variance of the trait.
	This might therefore lead to bias in \cref{eq:finalNarrow} and we shall discuss the partitioning of variance in the later sections.
	
	Nonetheless, \cref{eq:finalNarrow} was still useful for the understanding of the calculation of heritability.
	However, in the case of discontinuous trait (e.g. disease status) the calculation becomes more completed because the variance of the phenotype was dependent on the population prevalence.
	As \cref{eq:finalNarrow} does not account for the trait prevalence, it cannot be directly applied to discontinuous traits.
	In order to perform heritability estimation, we will need the concept of liability threshold model popularized by \cite{Falconer1965}.
	
	\section{Liability Threshold}
	\label{sec:liability}
	According the central limit theorem, if a phenotype is determined by a multitude of genetics and environmental factors with relatively small effect, then its distribution will likely follow a normal distribution as is the case of many quantitative traits\citep{Visscher2008}. % No, what if there is interaction between variables? Then it will break the CLT
	The variance of phenotype can therefore be calculated as the variance under the normal distribution.
	However, such is not the case for disease such as \glng{scz} where instead of having a continuous distribution of phenotype, only a dichotomous labeling of ``affected'' and ``normal'' were obtained.
	The variance of these phenotype were therefore more difficult to obtain.
	
	\citet{Falconer1965} proposed the liability threshold model, which suggesting that these discontinuous traits also follow a continuous distribution with an additional parameter called the ``liability threshold''.
	Under the liability threshold model, the discontinuous traits were also affected by combination of multitude of genetics and environmental factors, each with a small effects, as in the case of the continuous traits.
	The main difference was that the phenotype of an individual is determined by whether if the combined effects of these factors(``liability'') were above a particular threshold (``liability threshold'').
	So for example, in the case of \glng{scz}, only when an individual has a liability above the liability threshold will he/she be affected.
	
	One can then esimate the heritability of the discontinuous by comparing the mean liability of the general population when compared to the relatives of the affected individuals.	
	For example, if we consider a single threshold model of a dichotomous trait, where 
	\begin{align}
	T_G &= \text{Liability threshold of the general population}\notag\\
	T_R &= \text{Liability threshold of relatives of the index case} \notag\\
	q_G &= \text{Prevalence in the general population}\notag\\
	q_R &= \text{Prevalence in relatives of the index case}\notag\\
	L_a &= \text{Mean Liability of the index case} \notag
	\end{align}
	by assuming both the liability distribution of the general population and that of the relative of the index case both follows the standard normal distribution, we can align the two distribution with respect to $T_G$ and $T_R$. 
	We can then calculate the mean liability of the index case $L_a$ as $L_a=\frac{z_G}{q_G}$ where $z_G$ is the density of the normal distribution at the liability threshold $T_G$.
	Then we can express the regression of relative's liability on the liability of the index case as
	\begin{align}
	\beta &= \frac{T_G-T_R}{L_a}
	\label{eq:liability}
	\end{align}
	
	Thus, by applying \cref{eq:liability} to \cref{eq:finalNarrow}, we get
	\begin{align}
	h^2 =\frac{T_G-T_R}{L_ar}
	\end{align}
	
	% Then the application in schizophrenia
	% Then Twin studies 
	% Or maybe twin studies first, then the application in schziophrenia?
		
	\section{Twin Studies of Schizophrenia}
	% Need to go deeper into twin studies
	Now that we can deal with discontinuous traits, we shall come back to the limitation of \cref{eq:finalNarrow}.
	The key limitation of \cref{eq:finalNarrow} was its inability to discriminate the genetic factors from the shared environmental factors.
	Such problem arise as family not only shared some of their genes, but they also tends to shared some of the environmental factors such as diet. 
	In fact, this was the main reason for researchers to discord the argument that \glng{scz} was a genetic disorder.
	
	A classical adoption study carried out by \citet{HESTON1966} in 1966 set off to discriminate whether if the increased risk of \glng{scz} in relatives of \glng{scz} was caused by the shared environmental factors or the shared genetic factors. 
	An advantages of adoption studies was that if the child was separated from their family early after birth, then the shared environmental factors should be minimized, thus any resemblance between the parent and child should be driven mainly by the shared genetic factors.
	\citet{HESTON1966} collected data of 47 individuals born from a schizophrenic mother during the period from 1915 to 1947. 
	They were separated from their mother within three day of birth and were sent to a foster family. 
	50 matched control were also recruited to the study.
	It was observed that there was an increased risk of \glng{scz} in individual born to schizophrenic mother when compared to the control group even-though they were brought up in a different environment as that of their mother.
	This result suggested that \glng{scz} was likely driven by the shared genetic factors instead of the shared environmental factors.
	
	Despite the usefulness of adoption studies in delineating the effect of shared environment from the genetic factors, collection of adoption data were difficult. 
	Moreover, any prenatal influence such as alcohol abuse during pregnancy might confound the results.
	Therefore, an alternative way would be the twin studies using the relationship between the \gls{mz} and \gls{dz} twins.
	
	Theoretically, \gls{mz} twins should shared all their genetic components (both additive($A$) and non-additive($D$) genetic factors) and also their common environmental factors($C$) where the only difference between a twin pair would be the non-shared environmental factors($E$). 
	As for the \gls{dz} twins, they should also shared the same common environmental factors yet they only share $\frac{1}{2}$ of their additive genetic factors and $\frac{1}{4}$ of their non-additive genetic factors. 
	The non-shared environmental was also by definition not shared among the twins\citep{Rijsdijk2002}.
	Based on these assumptions, \cite{Falconer1996} derived the heritability as
	\begin{equation}
	h^2 = 2(\rho_{MZ}-\rho_{DZ})
	\end{equation}
	where $\rho_{MZ}$ and $\rho_{DZ}$ were the phenotype correlation between the \gls{mz} twins and \gls{dz} twins respectively.
	
	By combining Falconer's formula and the concept of liability threshold model, \citet{Gottesman01071967} estimated that the heritability of \glng{scz} to be $>60\%$ based on previously collected twin data, strongly suggesting \glng{scz} as a genetic disorder.
	The result was further supported by one of the landmark meta-analysis study conducted by \cite{Sullivan2003}.
	Based on data obtained from 12 published \glng{scz} twin studies, the authors found that although there was a non-zero contribution of environmental influence on liability of \glng{scz} ($11\%$,\gls{ci}=$3\%-19\%$), there was a much larger contribution from genetics ($81\%$, \gls{ci}=$73\%-90\%$), further supporting that \glng{scz} was largely mediated by the genetic factors.
	
	Such findings were not limited to twin-studies but were also reported in large scale population based studies.
	A recent large scale population based study in Sweden population\citep{Lichtenstein2009} also found that there was a large genetic contribution in \glng{scz} ($64\%$).
	Although the estimated heritability(64\%\citep{Lichtenstein2009} vs 81\%\citep{Sullivan2003}) differs between the two studies, they, there is no doubt that \glng{scz} is highly heritable, leading to the initiative of genetic research in \glng{scz}.
	
	
	
	\section{Genetic Analysis of Schizophrenia}
	\subsection{Genetic Architecture of Schizophrenia}
	\begin{wrapfigure}{R}{8cm}
		\centering
		\includegraphics[width=0.5\textwidth]{figure/lifeTimeMorbidRisk.png}
		\caption[Lifetime morbid risks of \glng{scz} in various classes of relatives of a proband]{Lifetime morbid risks of \glng{scz} in various classes of relatives of a proband.
			It was noted that the morbid risk of monozygotic (MZ) twins were only $48\%$, much lower than one would expect if \glng{scz} follows a Mendelian pattern.
			Reproduced with permission from journal\citep{Riley2006}. \label{fig:lifeMRscz}}
	\end{wrapfigure}
	Studies on estimation of heritability of \glng{scz} strongly support \glng{scz} as a genetic disorder.
	However, little was known about the mechanism of \glng{scz} nor the genetic architecture of the disorder. 
	All data from adoption studies, twin studies and family studies shown that \glng{scz} does not follow the Mendelian framework\cite{Gottesman01071967,Gottesman1982}.
	Specifically, shall \glng{scz} be a Mendelian disorder, then we would expect all \gls{mz} siblings of the proband to also suffer from \glng{scz}.
	However, the life time morbid risk of monozyogitc twins were only $48\%$(\cref{fig:lifeMRscz})\citep{gottesman1991schizophrenia}, making it unlikely for \glng{scz} to follow a Mendelian pattern.
	
	Based on these observations, \cite{Gottesman1967} proposed that \glng{scz} follows a polygenic model where disease phenotype were determined by the additive effects from multiple genes.
	Thus, \glng{scz} is a complex genetic disorder with complicated pattern of inheritance. 
	Their hypothesis was supported by the calculation of \cite{Risch1990a} by taking into account of different inheritance model and the life time morbid risk observed in relatives of affected individuals.
	
	Another interesting conclusion from the calculation of \citet{Risch1990a} was the effect size of individual locus. 
	By comparing the observed life time morbid risk and the calculated risk from different models, \citeauthor{Risch1990a} 	suggested that genetic models with a single locus with risk of 3.0 and with all other loci of small effect or models with two or three loci with risk of 2.0 were most consistent with the observed life time morbid risk of \glng{scz}.	\citep{Risch1990}.
	
	\citeauthor{Risch1990a}'s calculation provided an explanation for the early inconsistent findings of linkage studies in \glng{scz}\citep{Harrison2005}.
	As linkage studies were aimed to identify genetic variation of large effect size they failed to capture genetic loci with small effect size.
	It was therefore tempting to suggest that \glng{scz} only follows the ``common disease-common variant'' model, which stated that \glng{scz} should be mediated by large amount of common variants such as \glng{SNP}, each carries a small effect size.
	
	However, another possible hypothesis was that the variation mediating \glng{scz} were rare, therefore require a large sample size to detect. 
	The inconsistent results of the early linkage studies might be due to the inadequate sample size. 
	This lead to some researchers suggesting the ``common disease-rare variant'' hypothesis, which propose that \glng{scz} was mediated by a small amount of rare variants, each with a large effect size\citep{McClellan2007}.
	
	Nevertheless, success in genetic research of \glng{scz} remains limited.
	Only until the initiation of Human Genome Project and the technological advance resulted from the it does genetic research of \glng{scz} entered an era of success.

	\subsection{The Human Genome Project and HapMap Project}
	\glsreset{SNP}
	\glsreset{LD}
	In 1990, the Human genome project was initiated, aiming at constructing the first physical map of the human genome at per nucleotide resolution\citep{Lander2001}.
	The completion of the human genome project has opened up a new era of genetic research, allowing researchers to identify \glspl{SNP} on the human genome, which is one of the major source of genetic variation.
	
	Soon after the completion of the human genome project, the HapMap Project was initiated\citep{Consortium2005}, aiming to provide a genome-wide database of common human sequence variation such as \glspl{SNP} with \gls{maf} $\ge0.05$.
	More importantly was that the HapMap Project also provided a detailed \gls{LD} map of the human genome.
	
	\gls{LD} was of particular importance to genetic research for it was the non-random correlation of genotypes between 2 genetic locus. 
	\glspl{SNP} in high \gls{LD} were usually observed together in the human genome.
	When a large amount of \glspl{SNP} were in high \gls{LD} together, they form what was known as a \gls{LD} block.
	By performing association testing on \glspl{SNP} representing a \gls{LD} block(``tagging''), one can avoid the need of performing association on the whole genome, therefore reducing the cost of the experiment.
	This was the fundamental concept of \gls{GWAS} which tests the 
	which was now extensively used in the genetic research.
	
	\subsection{Genome Wide Association Study}
	In \gls{GWAS}, genome-wide genotyping array were commonly used to systematically detect genetic variants such as \gls{SNP} and \gls{cnv}.
	For quantitative traits, the association between the trait and frequency of the variants were calculated using methods such as linear regression.
	On the other hand, for dichotomous traits such as \glng{scz}, the frequency of the variants were compared between the case and control samples using methods such as chi-square test or logistic regression.
	Because of the problem of multiple testing, only variants with a p-value passing a genome wide threshold (p-value $\le5\times10^{-8}$) were considered significant.
	Another possible method to decide the significant threshold was to consider the ``effective number'' of tests\citep{Li2011} taking into consideration of \gls{LD} as not all tests in a \gls{GWAS} were independent of each other. 
	The power of the \gls{GWAS} were determined by the magnitude of effect, sample size, and required level of statistical significance(the false-positive, or type I, error rate)\citep{Purcell2003}.
	A large sample size and a large effect usually result in a larger power.
	
	\subsubsection{Single Nucleotide Polymorphism} 
	Despite the great promise from \gls{GWAS}, early \gls{GWAS} in \glng{scz} remain largely disappointing and were unable to identify any robust genetic markers associated with \glng{scz}.
	The failure of early \gls{GWAS} in \glng{scz} were mainly due to the relative small sample size of the studies, which result in low detection power.
	
	To overcome the problem of small sample size, large consortium were formed such that data from different research groups from different countries were combined, essentially providing a large sample size for the analysis.
	By 2014, the \Glng{scz} Working group of the \gls{pgc} has collected 34,241 \glng{scz} samples and 45,604 controls\citep{Ripke2014}.
	By combining the samples with those obtained by deCODE genetics, a total of 36,989 \glng{scz} samples and 113,075 controls were used for the largest meta analysis of \glng{scz}.
	In their study\citep{Ripke2014}, 128 linkage-disequilibrium-independent \glspl{SNP} were found to  exceeded the genome-wide significance(p-value $\le 5\times10^{-8}$), corresponding to 108 genetic loci.
	75\% of these loci contain protein coding genes and a further 8\% of these loci were within 20kb of a gene. 
	It was found that genes involved in glutamatergic neurotransmission (e.g. \textit{GRM3}, \textit{GRIN2A} and \textit{GRIA1}), synaptic plasticity and genes encoding the voltage-gated calcium channel subunits (e.g. \textit{CACNA1C}, \textit{CACNB2} and \textit{CACNA1I}) were among the genes associated within these loci.
	Importantly, \textit{DRD2}, the target of all effective anti-psychotic drug were also associated with \glng{scz}.
	This result converges with existing knowledge of \textit{DRD2} being involved in the pathology of \glng{scz}, supported by multiple lines of research\citep{Talkowski2007}.
	\begin{figure}
		\centering
		\caption[Enrichment of enhancers of SNPs associated with Schizophrenia]{Enrichment of enhancers of SNPs associated with \glng{scz}. 
			It was observed that the largest enrichment were in cell lines related to the brain and in tissues with important immune functions. 
			Graphs reproduced with permission from the journal.\citep{Ripke2014}}
		\includegraphics[height=\textwidth]{figure/pgc_enrichment_tissue.jpg}
		\label{fig:pgcEnrich}
	\end{figure}
	It was further demonstrated that \glng{scz} association were significantly enriched at enhancers active in brain and enriched at enhancers active in tissues with important immune functions(\cref{fig:pgcEnrich})\citep{Ripke2014}.
		
	The enrichment of immune related enhancers remains significant even after the removal of \gls{mhc} region from the analysis, provided further genetic support of the involvement of the immune system in the etiology of \glng{scz}.
	Because of its role in neural development\citep{Zhao1998,Deverman2009}, it is likely that the perturbation in the immune system might disrupt the brain development, therefore increasing the risk of \glng{scz}.
	Indeed, studies on \gls{mia} has demonstrated that cytokine imbalance might predispose individual to \glng{scz}\citep{Meyer2009}. 
	
	\subsubsection{Copy Number Variation}
	\glsreset{cnv}
	Another important arm of genetic research in \glng{scz} was to identify \gls{cnv} associated with \gls{scz}.
	\gls{cnv} were classified as segment of DNA that is 1kb or larger and that is present at a different copy number when compared to the reference genome, usually in the form of insertion, deletion or duplication\citep{Feuk2006}.
	Due to the length of these variants, the \gls{cnv} might contain the entire genes and their regulatory regions which might in turn contribute to significant phenotypic differences\citep{Feuk2006}.
	
	To identify robust association between \gls{cnv} and \glng{scz}, \cite{Szatkiewicz2014} conducted a \gls{GWAS} for \gls{cnv} association with \glng{scz} used the Swedish national sample (4,719 \glng{scz} samples and 5,917 controls).
	In their study, they were able to association between \glng{scz} and \gls{cnv} such as 16p11.2 duplications, 22q11.2 deletions, 3q29 deletions and 17q12 duplications were identified.
	Through the gene set association analysis, calcium channel signaling and binding partners of the fragile X mental retardation protein were found to be associated with these \gls{cnv}\citep{Szatkiewicz2014}.
	Interestingly, the calcium channel signaling were also enriched in the \gls{pgc} \gls{GWAS} on \gls{SNP} association, suggesting that the variants were converging on similar set of pathway or gene sets. 
	
	% I need to state rare and large effect because LDSC cannot do that 
	Unlike the result form the \gls{GWAS} on \gls{SNP} data, the \gls{cnv} identified were rare($\le12$ in 4,719 samples) and has a relative large effect (e.g. 22q11 deletion has an odd ratio of 16.32\citep{Szatkiewicz2014}). 
	The results from the \gls{SNP} \gls{GWAS} supports the ``common disease-common variant'' model whereas the \gls{GWAS} on \gls{cnv} supports the ``common disease-rare variant'' model, illustrating the complex genetic model behind the etiology of \glng{scz}.
	
	Although the \gls{GWAS} in \glng{scz} seems to return a lot of interesting results, the question remains: How much of the known genetic risk factors associated explain the disease risk of \glng{scz}?
	To answer these question, we need to estimate the heritability based on the \gls{GWAS} data. 
	However, in order to obtain the large volume of data, most of the samples were not relatives. 
	How can one estimate the heritability based only on the genetic data of the general population instead of family or twin data?
	
	\subsection{\glng{gcta}}
	Unlike family based data, the relationship between the samples were unknown. 
	Yet in a typical \gls{GWAS}, the genotype of each individuals were known.
	The ``genetic distance'' between two individual will provide an estimate of their relationship, thus allowing the calculation of heritability.
	\citet{Yang2011} use the concept of genetic distance to calculate the \gls{grm} to represent the relationship between individuals.
	The \gls{grm} were then used in the restricted maximum likelohood analysis(REML) to estimate the heritability of the trait\citep{Yang2011}.
	This was implemented in \gls{gcta} and were now wildly used in the estimation of heritability on \gls{GWAS} data.
	
	The problem with \gls{gcta} was that it require the genotype data to estimate the heritability.
	However, for complex disease like \gls{scz}, the data were usually obtained from multiple data source.
	Because of privacy issues, usually only the test statistic were shared among the research groups and only meta analysis were performed.
	Given there was no raw genotype data, it is impossible to calculate the \gls{grm}, thus making the use of \gls{gcta} impossible.
	  
	\subsection{\glng{ldsc}}
	Sometimes, in a \gls{GWAS} study, one can observe an general inflation of test statistics. 
	It was usually considered to be contributed to the presence of confounding factors such as population stratification under the assumption that most of the \glspl{SNP} should have no association to the disease.
	It was therefore a common practice for one to perform the \gls{gc} on the \gls{GWAS} results\citep{Zheng2006}.
	
	The problem of the \gls{gc} was that the basic assumption of a small number of causal \glspl{SNP} might not be true. 
	Through careful simulation, \citet{Yang2011b} demonstrated that in the absence of population stratification and other form of technical artifacts, the presence of polygenic inheritance can also inflate the test statistic\citep{Yang2011b}.
	More importantly, they observed that the magnitude of inflation was determined by the \emph{heritability}, the \gls{LD} structure, sample size and the number of causal \glspl{SNP} of the trait.

	Following on this observation, \citet{Bulik-Sullivan2015} developed the \gls{ldsc}.
	The fundamental concept of \gls{ldsc} was that the more genetic variant a \gls{SNP} tag, the more likely for it to be able to tag a causal variant; 
	whereas population stratification and cryptic relatedness should not be associated with \gls{LD}.
	The number of genetic variants tagged by a \gls{SNP}$_j$ ($l_j$)(\gls{LD} score) was then defined as the sum of $r^2$ of the $k$ \glspl{SNP} within a 1cM window of \gls{SNP}$_j$:
	\begin{equation}
	l_j = \sum_kr^2_{jk}
	\label{eq:ldScore}
	\end{equation}
	
	The expected $\chi^2$ of \gls{SNP}$_j$ was then defined as a function of the \gls{LD} score ($l_j$), the number of samples ($N$), the number of \glspl{SNP} in the analysis($M$), the contribution of confounding factors ($a$) and most importantly, the heritability ($h^2$):
	\begin{equation}
	\mathrm{E}[\chi^2_j | l_j] = \frac{Nl_jh^2}{M}+Na+1
	\label{eq:fullLDSC}
	\end{equation}
	If one express the \gls{LD} score and the $chi^2$ as vectors $\boldsymbol{L}$ and $\boldsymbol{\chi^2}$ respectively, \cref{eq:fullLDSC} becomes a regression of the $\chi^2$ against the \gls{LD} score:
	\begin{equation}
	\boldsymbol{\chi^2}= \frac{N}{M}\boldsymbol{L}h^2+Na+1
	\label{eq:ldReg}
	\end{equation}
	
	As a result of that, the heritability $h^2$ will be the slope of the regression and the intercept minus one will represent the mean contribution of the confounding bias such as those of population stratification. 
	Thus, \cref{eq:ldReg} can be used for the estimation of heritability given only the test statistics and the population \gls{LD} were provided. 
	
	
	Using data from \citet{Ripke2014}, and applying the liability threshold adjustment, \citet{Bulik-Sullivan2015} estimated the heritability of \glng{scz} should be 0.555 with \gls{se} of 0.008.
	The estimated heritability was lower than what was previously estimated from population based study(64\%\citep{Lichtenstein2009}) and twin studies(81\%\citep{Sullivan2003}).
	Possible reasons of such discrepancies might be that in \citet{Ripke2014}'s study, only \glspl{SNP} data were collected. 
	From \citet{Szatkiewicz2014}, it was clearly demonstrated that other than \glspl{SNP}, \glspl{cnv} were also associated with \glng{scz}.
	By ignoring \glspl{cnv} in the estimation of heritability, the estimation of \citet{Bulik-Sullivan2015} would only provided a lower bound of heritability estimated.
	Another possibility of the``missing'' heritability can be due to interaction between the genetic and environmental factors. 
	Although previous studies\citep{Gottesman01071967} suggested that the non-additive genetic factors were unlikely to contribute to \glng{scz}, the possibility of involvement of gene-environmental interaction $G\times E$ were not ruled out.
	Indeed, in the adoption study conducted by \citet{Tienari2004}, it was found that individuals with higher genetic risk were significantly more sensitive to ``adverse'' vs ``healthy'' rearing patterns in adoptive families than are adoptees at low genetic risk\citep{Tienari2004}, providing support to a possible interaction between genetic and environmental factors.
	Therefore, in order to account for the ``missing'' heritability, one might need to consider genetic variations other than \glspl{SNP} and might need to take into consideration of the $G\times E$ interaction.
	
	Nonetheless, the heritability estimation from \citet{Ripke2014} were still encouraging, as for the first time in genetic research of \glng{scz}, a large portion of heritability of \glng{scz} were finally identified.
	This permit the genetic research of \glng{scz} to move beyond statistical association and focus on the functional basis of the genetic susceptibility locus of \glng{scz}.
	
	\subsection{Partitioning of Heritability of Schizophrenia}
	\subsectionmark{Partitioning of Heritability}
	Traditionally, functional enrichment analysis in \gls{GWAS} only take into account of \glspl{SNP} that passed the genome wide significance threshold. 
	However, for complex traits such as that of \glng{scz}, much fo the heritability might lies in \glspl{SNP} that do not reach genome wide significance threshold at the current sample size.
	For example, in 2013, only 13 risk loci were detected using 13,833 \glng{scz} samples and 18,310 controls \citep{Ripke2013}. 
	When the sample size increased to 34,241 \glng{scz} samples and 45,604 controls in 2014, 108 risk loci were identified\citep{Ripke2014}. 
	Thus, if one only consider the significant loci, risk loci that have not reach genome wide significance threshold might be ignored from the analysis, decreasing the power of the functional enrichment analysis.
	
	Unlike traditional functional enrichment analysis, \gls{ldsc} uses information from all \glspl{SNP} and taking into account of the \gls{LD} structure to partition heritability into different functional categories. 
	Thus should be more powerful when compared to traditional analysis and should help to provide useful insight into the disease etiology of \glng{scz}.

	\citet{Finucane2015} used data from \citet{Ripke2014} and functional categories derived from the ENCODE annotation\citep{ENCODEProjectConsortium2012}, the NIH Roadmap Epigenomics Mapping Consortium annotation\citep{Bernstein2010} and other studies\citep{Finucane2015}, it was found that the brain cell types were most enriched in \glng{scz}, especially those related to the \gls{cns}.
	Of all the functional categories, the most enriched category in \glng{scz} was the H3K4me3 mark in the fetal brain(\cref{tab:cellTypeScz}). 
	As H3K4me3 was mostly linked to active promoters, it was likely for genes that were active in fetal brain (e.g genes related to brain development) to be associated with \glng{scz}, supporting the idea of \glng{scz} as an neuro-developmental disorder. 
	
	Moreover, it was also observed that the second most enriched cell types were those related to immunity.
	Undoubtedly, the \gls{cns} and the immune system have an important role in the disease etiology of \gls{scz}. 

	\begin{singlespace}
	\begin{longtable}{p{6cm}rrr}
		%\begin{tabular}{rrrr}
			\toprule
			Cell type & cell-type group & Mark  & P-value \\
			\midrule
			Fetal brain** & CNS   & H3K4me3 & $3.09\times 10^{-19}$ \\
			Mid frontal lobe** & CNS   & H3K4me3 & $3.63\times 10^{-15}$ \\
			Germinal matrix** & CNS   & H3K4me3 & $2.09\times 10^{-13}$ \\
			Mid frontal lobe** & CNS   & H3K9ac & $5.37\times 10^{-12}$ \\
			Angular gyrus** & CNS   & H3K4me3 & $1.29\times 10^{-11}$ \\
			Inferior temporal lobe** & CNS   & H3K4me3 & $1.70\times 10^{-11}$ \\
			Cingulate gyrus** & CNS   & H3K9ac & $5.37\times 10^{-11}$ \\
			Fetal brain** & CNS   & H3K9ac & $5.75\times 10^{-11}$ \\
			Anterior caudate** & CNS   & H3K4me3 & $2.19\times 10^{-10}$ \\
			Cingulate gyrus** & CNS   & H3K4me3 & $4.57\times 10^{-10}$ \\
			Pancreatic islets** & Adrenal/Pancreas & H3K4me3 & $2.24\times 10^{-09}$ \\
			Anterior caudate** & CNS   & H3K9ac & $3.16\times 10^{-9}$ \\
			Angular gyrus** & CNS   & H3K9ac & $4.68\times 10^{-9}$ \\
			Mid frontal lobe** & CNS   & H3K27ac & $7.94\times 10^{-9}$ \\
			Anterior caudate** & CNS   & H3K4me1 & $1.20\times 10^{-8}$ \\
			Inferior temporal lobe** & CNS   & H3K4me1 & $3.72\times 10^{-8}$ \\
			Psoas muscle** & Skeletal Muscle & H3K4me3 & $4.17\times 10^{-8}$ \\
			Fetal brain** & CNS   & H3K4me1 & $6.17\times 10^{-8}$ \\
			Inferior temporal lobe** & CNS   & H3K9ac & $9.33\times 10^{-8}$ \\
			Hippocampus middle** & CNS   & H3K9ac & $9.33\times 10^{-7}$ \\
			Pancreatic islets** & Adrenal/Pancreas & H3K9ac & $1.62\times 10^{-6}$ \\
			Penis foreskin melanocyte primary** & Other & H3K4me3 & $2.09\times 10^{-6}$ \\
			Angular gyrus** & CNS   & H3K27ac & $2.34\times 10^{-6}$ \\
			Cingulate gyrus** & CNS   & H3K4me1 & $2.82\times 10^{-6}$ \\
			Hippocampus middle** & CNS   & H3K4me3 & $2.82\times 10^{-6}$ \\
			CD34 primary** & Immune & H3K4me3 & $4.68\times 10^{-6}$ \\
			Sigmoid colon** & GI    & H3K4me3 & $5.01\times 10^{-6}$ \\
			Fetal adrenal** & Adrenal/Pancreas & H3K4me3 & $6.31\times 10^{-6}$ \\
			Inferior temporal lobe** & CNS   & H3K27ac & $8.32\times 10^{-6}$ \\
			Peripheralblood mononuclear primary** & Immune & H3K4me3 & $9.33\times 10^{-6}$ \\
			Gastric** & GI    & H3K4me3 & $1.17\times 10^{-5}$ \\
			Substantia nigra* & CNS   & H3K4me3 & $1.95\times 10^{-5}$ \\
			Fetal brain* & CNS   & H3K4me3 & $2.63\times 10^{-5}$ \\
			Hippocampus middle* & CNS   & H3K4me1 & $3.31\times 10^{-5}$ \\
			Ovary* & Other & H3K4me3 & $6.46\times 10^{-5}$ \\
			CD19 primary (UW)* & Immune & H3K4me3 & $7.08\times 10^{-5}$ \\
			Small intestine* & GI    & H3K4me3 & $8.51\times 10^{-5}$ \\
			Lung* & Cardiovascular & H3K4me3 & $1.17\times 10^{-4}$ \\
			Fetal stomach* & GI    & H3K4me3 & $1.29\times 10^{-4}$ \\
			Fetal leg muscle* & Skeletal Muscle & H3K4me3 & $1.51\times 10^{-4}$ \\
			Spleen* & Immune & H3K4me3 & $1.70\times 10^{-4}$ \\
			Breast fibroblast primary* & Connective/Bone & H3K4me3 & $2.04\times 10^{-4}$ \\
			Right ventricle* & Cardiovascular & H3K4me3 & $2.14\times 10^{-4}$ \\
			CD4+ CD25- Th primary* & Immune & H3K4me3 & $2.19\times 10^{-4}$ \\
			CD4+ CD25- IL17- PMA Ionomycin stim MACS Th sprimary* & Immune & H3K4me1 & $2.19\times 10^{-4}$ \\
			CD8 naive primary (UCSF-UBC)* & Immune & H3K4me3 & $2.24\times 10^{-4}$ \\
			Pancreas* & Adrenal/Pancreas & H3K4me3 & $2.34\times 10^{-4}$ \\
			CD4+ CD25- Th primary* & Immune & H3K4me1 & $2.75\times 10^{-4}$ \\
			CD4+ CD25- CD45RA+ naive primary* & Immune & H3K4me1 & $2.75\times 10^{-4}$\\
			Colonic mucosa* & GI    & H3K4me3 & $3.24\times 10^{-4}$ \\
			Right atrium* & Cardiovascular & H3K4me3 & $3.31\times 10^{-4}$ \\
			Fetal trunk muscle* & Skeletal Muscle & H3K4me3 & $3.39\times 10^{-4}$ \\
			CD4+ CD25int CD127+ Tmem primary* & Immune & H3K4me3 & $3.47\times 10^{-4}$ \\
			Substantia nigra* & CNS   & H3K9ac & $3.63\times 10^{-4}$ \\
			Placenta amnion* & Other & H3K4me3 & $4.17\times 10^{-4}$ \\
			Breast myoepithelial* & Other & H3K9ac & $5.50\times 10^{-4}$ \\
			CD8 naive primary (BI)* & Immune & H3K4me1 & $5.75\times 10^{-4}$ \\
			Substantia nigra* & CNS   & H3K4me1 & $6.61\times 10^{-4}$ \\
			Cingulate gyrus* & CNS   & H3K27ac & $7.94\times 10^{-4}$ \\
			CD4+ CD25- CD45RA+ naive primary* & Immune & H3K4me3 & $8.71\times 10^{-4}$ \\
			\bottomrule
		%\end{tabular}%
		\caption[Enrichment of Top Cell Type of Schizophrenia]{Enrichment of Top Cell type of Schizophrenia.
			* = significant at False Discovery Rate $<$ 0.05.
			** = significant at p $<$ 0.05 after correcting for multiple hypothesis. 
			Reproduce with permission from Journal.\citep{Finucane2015}}
		\label{tab:cellTypeScz}%
	\end{longtable}%
	\end{singlespace}
	
	\subsection{Genetic Correlation}
	Another very important application of \gls{ldsc} is that it allow one to identify the genetic correlation between traits\citep{Bulik-Sullivan2015a}. 
	The genetic correlation can be used as an genetic analogue to co-morbidity, thus allowing deeper understanding to the etiology of the traits.
	Above all, genetic correlation was important in studying the treatment response. 
	It has been observed that there was an increased prevalence of anxiety, depression and substance abuse in \glng{scz}\citep{Buckley2009}. 
	These co-morbidity were generally associated with more severe psychopathology and with poorer outcome\citep{Buckley2009}.
	A deeper understanding of possible co-morbidity between different traits and \glng{scz} might provide insight not only to the disease etiology of \glng{scz}, it might even provide important information in possible treatment options for \glng{scz}. 
	Using breast cancer as an example, it was found that patients with comorbidity had poorer survival than those without comobidity\citep{Sogaard2013} and it was suggested that by treating the comorbid diseases, one might be able to delay mortality in breast cancer patients\citep{Ording2013}.
		
	By applying their method to 25 different phenotypes, \citet{Bulik-Sullivan2015a} shown that \glng{scz} has significant genetic correlation with bipolar disorder, major depression and more surprisingly, anorexia nervosa.
	Previous studies have always suggest there to be an co-morbidity between \glng{scz} and bipolar disorder \citep{Lichtenstein2009,Purcell2009,Buckley2009}.
	Similarly, it was not uncommon for \glng{scz} to display depressive symptoms\citep{Buckley2009}. 
	It was even observed that individuals at high risk and ultrahigh risk for developing \glng{scz} have generally demonstrated a significant degree of depressive symptoms prior to and during the emergence of psychotic symptoms, suggesting a close relationship between \glng{scz} and depression. 
	
	On the other hand, the genetic correlation between \glng{scz} and anorexia nervosa were slightly unexpected for there has been a lack of study in the co-morbidity between eating disorder and \glng{scz}. 
	Nonetheless, this finding raises the possibility of similarity between anorexia and nervosa.
	
	\section{Antipsychotics}
	Despite the success in the genetic research of \gls{scz}, an effective cure of \glng{scz} was yet to be found.
	Currently, the main treatment method for \glng{scz} was the use of antipsychotic drugs yet there was a large variability between individuals in their response to the drugs.
	Some might even suffer from adverse side effects such as agranulocytosis.
	Thus, it is important to administrate the ``correct'' drug for each individual.
	However, there was an lack of understanding of the factors influencing the drug response, making it difficult, if not impossible, to develop a robust diagnostic test for selecting the most appropriate treatment for individuals.
	
	% There was an important need for the study
	% Need an direction
	% Heritability estimation of drug response
	% Lead to genetic testings
	% Candidate genes, GWAS no consistent results
	% Questions regarding the heritability of treatment response?
	% Then talk about the current thesis.
	
	\subsection{Pharmacogenetics and Pharmacogenomics}
	\section{Chapter Summaries}
	
	\include{heritability_estimation}
	
	\chapter{Heritability of Schizophrenia}
	\section{Introduction}
	Apply Heritability estimation to the schizophrenia data.
	The genetic correlation and partitioning of heritability
	No one worked on linking schizophrenia with brain development directly?
	% talk about the current research on schizophrenia?
	% the overall heritablity estimation
	% The genetic correlation done by the LDSC. 
	% The theory of brain development 
	% How the co-expression network works
	% Drug response?
	\section{Heritability Estimation}
	This will be a very simple section, focused on how to perform the heritability estimation on \acrfull{scz}.
	Should also tokenize the heritability into subcategories (e.g. immune, neuron, etc)
	%Should not put too much weight into it, otherwise it will be a direct copy of LDSC. Won't really add much power. 
	
	
	\subsection{Methodology}
	\subsection{Result}
	\section{Brain development and Schizophrenia}
	\sectionmark{Brain development}
	Here we will perform the WGCNA and brain development network.
	Seeing how the whether if any brain development network were enriched with SNPs that explain the variance of phenotype
	%Instead, we should put most focus here as no one has done it before
	%Also descript brainspan here
	\subsection{Methodology}
	\subsubsection{Sample Quality Controls}
	We obtain the developmental transcriptome data from BrainSpan (\url{http://www.brainspan.org/}). 
	A total of 56 samples with different age were provided by BrainSpan with an average of 2.2 samples per age.
	
	Studies suggested Hippocampus\citep{Velakoulis2006,Nugent2007}, Amygdala and Striatum\citep{Simpson2010} are brain regions involved in the etiology of schizophrenia. 
	Therefore, we focus on building the gene co-expression network of hippocampus, amygdala and striatum in this study
	It is worth noting that the Pre-frontal Cortex is also important for schizophrenia. 
	However, as there isn't a well defined pre-frontal cortex samples from BrainSpan, we did not include the pre-frontal cortex in the current study.
	RNA Sequencing data of the brain regions were obtained from BrainSpan and undergo a series of quality control before the construction of the network. 
	 
	For each sample age, when there are more than one samples, we select the sample with a dissection score $\ge3$ and an \gls{rin} $\ge7$. 
	As some developmental stage only got 1 sample passing the quality check, we limit each developmental stage to have a maximum of 1 sample such that the final network will not be driven by a particular developmental stage. 
	If multiple samples passed through the quality check threshold, we will prefer sample with higher dissection score. 
	Shall multiple samples have the same dissection score, we will select the one with the highest \gls{rin}. 
	And if the samples have the same dissection score and \gls{rin} value, we will randomly select one for the network construction.
	
	After performing the quality control, a total of 16, 18 and 15 samples were selected for hippocampus, amygdala and striatum respectively.
	The sample age ranged from \gls{gd}8 to 23 years old representing the fetal developmental stage till the age of onset of schizophrenia.
	
	\subsubsection{Normalization of data}
	The RNA Sequencing data were represented as \gls{rpkm} values. 
	Genes with a low \gls{rpkm} can usually be a result from technical or biological noise\citep{Hart2013}.
	To reduce noise in the final model, genes with a mean \gls{rpkm} $< 1$ in all samples were discarded. 
	The \gls{rpkm} were then log transformed as instructed by the manual of \gls{wgcna}\citep{Langfelder2008}.
	
	As there are insufficient samples for the construction of gene co-expression network for individual sample age, we try to construct networks with genes co-expressed through all sample stage. 
	This is achieved by taking the standardized log$_2$ \gls{rpkm} across sample age such that all genes has a mean of 0 and standard deviation of 1.
	
	At the end, there were 17,168 genes, 17,038 genes and 17,166 genes passing through the quality threshold and were used for the construction of co-expression network in hippocampus, amygdala and striatum respectively. 
	 
	\subsubsection{Network Construction}
	\gls{wgcna} (ver 1.47) were used for the construction of gene co-expression network\citep{Langfelder2008}. 
	The \emph{blockwiseModules} function, using Biweight Midcorrelation for the construction of correlation matrix and a restriction of minimum network size of 30. 
	For the construction of gene co-expression networks in hippocampus, the soft-power threshold were set to 15 where it is the first threshold value which has $R^2 > 0.8$ (0.817) and the $R^2$ is saturated\citep{Zhang2005}.%(\cref{fig:softpowerThreshold})
	As for striatum, the soft-power threshold were set to 20. 
	Again, this is the first threshold value with $R^2 >0.8$ (0.879) and where the $R^2$ is saturated.
	
	On the other hand, for amygdala, soft-power threshold were set to 9 which is the first threshold for $R^2$ to reach saturation. 
	However, with a soft-power threshold of 9, the $R^2$ were only 0.776, which is lower than the recommended 0.8 threshold.
	The reason behind this decision was that the first soft-power threshold to have $R^2 > 0.8$ is 30.
	Under this threshold, the mean connectivity of the resulting networks will be around 23.6 with a median connectivity of 2.51.
	Such level of connectivity will likely yield networks that are too small to useful.
	If one would like to satisfy both requirement of threshold selection, a threshold $>30$ are likely required and any networks constructed will likely to be small.
	As a result of that, we select threshold of 9 where networks with reasonable size can be constructed.
	
	%\begin{figure}
	%	\caption[Soft-power threshold selection]{Soft-power threshold selection. A soft-power of 13 were selected as it is the first threshold value having $R^2 > 0.8$ (0.817) and where the $R^2$ is saturated.}
	%	\centering
	%	\scalebox{.8}{\includegraphics{figure/SoftpowerThreshold.png}}
	%	\label{fig:softpowerThreshold}
	%\end{figure}
	
	\subsubsection{Expression correlation with Age}
	The co-expression network constructed with the standardized gene expression value will contains genes that co-express in all sample age.
	However, this does not necessary suggest the expression of these genes are correlated with the sample age.
	To identify gene co-expression networks with expressions correlated with the sample age, we performed a correlation analysis between the module eigen-genes and the sample age. 
	Network eigen-genes were calculated as the first \gls{pc} of expressions of the genes within individual networks using the \emph{moduleEigengenes} function from \gls{wgcna}. 
	Age were represented as month from conception such that 8 post-conception week will be represented as 2; 4 months will be represented as 10 and 12 years will be represented as 154 etc. 
	Finally, correlation between age and network eigen-gene expression were calculated pearson correlation.
	
	\subsubsection{Functional Annotation}
	\gls{GO} based enrichment analysis of the significant module was performed using GOrilla\citep{Eden2009}.
	Genes within the networks were provide as the target gene lists and all the genes passed quality controls were used as the background gene list.
	As \gls{GO} terms tends to be redundant and overlaps with each other, it will aid the interpretation of \gls{GO} results based by clustering and reducing the \gls{GO} terms based on their similarity. 
	Thus, \gls{GO} enrichment results were summerized by REViGO\citep{Supek2011} and significant representative \gls{GO} terms were obtained.
	
	\subsubsection{Associate Co-expression network with \glsentryshort{pgc} schizophrenia data}
	The co-expression networks were built from normal samples and should not be representative of the brain expression pattern in schizophrenia patients.
	it is however interesting to see if the co-expression networks were disrupted in schizophrenia patient.
	To test whether if the gene co-expression networks contain genes that are jointly associated with schizophrenia, we first use \gls{MAGMA}\citep{DeLeeuw2015}(version v1.03) to compute the gene-base p-value from the \gls{SNP} wise p-value obtained from \gls{pgc}. 
	Gene-set enrichment analysis were then performed on networks that were significantly correlated with developmental age. 
	As we were only interested in whether if the genes within the networks were jointly associated with schizophrenia, we only focus on the result of the self-contained gene set analysis and ignore the result from competitive analysis.
	
	\subsubsection{Partitioning of Heritability}
	
	\subsection{Result}
	\subsubsection{Co-Expression Network}
	A total of 35 networks were constructed based on the hippocampus samples with a mean network size of 421.6.
	On the other hand, 28 networks were constructed for amygdala with mean network size of 591.86.
	Finally, 25 networks with mean size of 494.52 were constructed from the striatum samples.
	
	Of the all the networks constructed, only one network from hippocampus(\cref{tab:hipModSig}) and three networks from amygdala(\cref{tab:amyModSig}) were significantly correlated with sample age after bonferroni correction threshold (p-value $<0.00143$ for hippocampus, p-value $<0.00179$ for amygdala and p-value $<0.002$ for striatum) .
	\begin{table}
		\centering
		\caption[Correlation of sample age with the module eigen gene]{Correlation of sample age with the module eigen gene. 
			Module eigen-gene was defined as the first \gls{pc} of genes within the module. 
			After correcting for multiple testing, only the black module was considered as significantly correlated with the sample age.}
		\subfloat[Hippocampus]{
			\begin{tabular}{rrr}
				\toprule
				& Correlation & Pvalue \\
				\midrule
				black & 0.804653 & 0.000171 \\
				blue  & -0.61648 & 0.010981 \\
				red   & -0.60207 & 0.013595 \\
				darkred & -0.59137 & 0.015833 \\
				greenyellow & -0.56995 & 0.021168 \\
				yellow & 0.567828 & 0.021763 \\
				darkgrey & -0.55246 & 0.026474 \\
				saddlebrown & -0.52983 & 0.034783 \\
				turquoise & -0.51371 & 0.041809 \\
				purple & -0.46788 & 0.067606 \\
				darkolivegreen & -0.41272 & 0.112122 \\
				sienna3 & -0.39535 & 0.129604 \\
				darkturquoise & 0.386541 & 0.139154 \\
				darkorange & 0.384966 & 0.140912 \\
				darkmagenta & 0.375586 & 0.151688 \\
				brown & 0.366095 & 0.163144 \\
				tan   & -0.36522 & 0.164229 \\
				pink  & 0.348979 & 0.18524 \\
				magenta & -0.32559 & 0.218473 \\
				midnightblue & -0.29168 & 0.273014 \\
				lightgreen & 0.289921 & 0.276056 \\
				paleturquoise & -0.28045 & 0.29276 \\
				white & 0.27727 & 0.29849 \\
				orange & 0.19607 & 0.466754 \\
				steelblue & 0.17355 & 0.520357 \\
				skyblue & 0.145869 & 0.589857 \\
				lightyellow & -0.11665 & 0.667028 \\
				green & -0.09882 & 0.715786 \\
				violet & -0.08757 & 0.747076 \\
				lightcyan & -0.0656 & 0.809257 \\
				cyan  & -0.06441 & 0.812661 \\
				darkgreen & -0.03914 & 0.885582 \\
				salmon & 0.038727 & 0.886769 \\
				royalblue & -0.03785 & 0.889314 \\
				grey60 & 0.03119 & 0.908709 \\
				\bottomrule
				\label{tab:hipModSig}%
			\end{tabular}%
		}
		\qquad%
		\subfloat[Amygdala]{
			\begin{tabular}{rrr}
				\toprule
				& Correlation & P-value \\
				\midrule
				tan   & 0.849999 & $7.96\times 10^{-6}$ \\
				yellow & -0.757 & $2.76\times 10^{-4}$ \\
				pink  & -0.68541 & $1.69\times 10^{-3}$ \\
				greenyellow & -0.67831 & $1.97\times 10^{-3}$ \\
				red   & -0.64532 & $3.83\times 10^{-3}$ \\
				turquoise & -0.59771 & $8.80\times 10^{-3}$ \\
				lightyellow & -0.56347 & 0.0149 \\
				brown & 0.548516 & 0.0184 \\
				darkgreen & -0.46366 & 0.0526 \\
				blue  & -0.4604 & 0.0545 \\
				purple & -0.44182 & 0.0664 \\
				darkgrey & -0.39065 & 0.109 \\
				orange & -0.36966 & 0.131 \\
				white & 0.28737 & 0.248 \\
				darkred & 0.283247 & 0.255 \\
				black & 0.271383 & 0.276 \\
				salmon & -0.24203 & 0.333 \\
				skyblue & 0.207071 & 0.410 \\
				cyan  & 0.18778 & 0.456 \\
				lightgreen & 0.166495 & 0.509 \\
				grey60 & 0.15156 & 0.548 \\
				midnightblue & 0.136078 & 0.590 \\
				magenta & -0.13459 & 0.594 \\
				darkturquoise & 0.129954 & 0.607 \\
				lightcyan & 0.090241 & 0.722 \\
				darkorange & -0.05166 & 0.839 \\
				green & -0.04745 & 0.852 \\
				royalblue & 0.020456 & 0.936 \\
				\bottomrule
				\label{tab:amyModSig}%
			\end{tabular}%
		}
	\end{table}
	
	By plotting the mean expression of each network against the sample age, one can inspect how the dynamic of the network changes across different developmental stage.
	Thus, mean expression of all the genes within the significant networks were calculated for all amygdala (n=33) and hippocampus (n=32) samples from BrainSpan.
	The mean \gls{rpkm} values were then log$_2$ transformed and plot against the sample age where a line of bests fit was calculated using the \emph{stat\_smooth} with the loess function from R package \emph{ggplot2}(version 1.0.1). (\cref{fig:allMod}).
	
	The expression pattern observed were intriguing where there both the ``black''(\cref{fig:blackMod}) and ``tan'' (\cref{fig:tanMod}) networks have mean gene expression level increase as development progress and reaches its peak at around late adolescence ($\approx 18-21$), concurring with the onset age of schizophrenia.
	Similarly, an inverse pattern were observed with the ``yellow'' network where its mean expression was highest during fetal development and drop steadily to its lowest around late adolescence and increase again afterwards(\cref{fig:yellowMod}).
	
	The expression pattern of the ``black'' and ``tan'' networks are of particular interest as they follow the inverted ``U'' shape trajectory of the grey matter volumn observed in previous studies\citep{Gogtay2011}, suggest that they might have a role in mediating brain development. 
	\begin{figure}
		\caption[Mean Gene Expression across developmental age]{Mean Gene Experssion across developmental age.
			Mean \gls{rpkm} values of genes in the significant modules were plot with respect to the sample age.
			A loess smoothing curve was also plotted. 
			%Might want to talk somemore about it
		}
		\centering
		\subfloat[``Black'' Network from Hippocampus]{
			\scalebox{.4}{\includegraphics{figure/network/hip_network.png}}
			\label{fig:blackMod}
		}
		\subfloat[``Tan'' Network from Amygdala]{
			\scalebox{.4}{\includegraphics{figure/network/amy_tan_network}}
			\label{fig:tanMod}
		}\\
		\subfloat[``Pink'' Network from Amygdala]{
			\scalebox{.4}{\includegraphics{figure/network/amy_pink_network}}
			\label{fig:pinkMod}
		}
		\subfloat[``Yellow'' Network from Amygdala]{
			\scalebox{.4}{\includegraphics{figure/network/amy_yellow_network}}
			\label{fig:yellowMod}
		}
		\label{fig:allMod}
	\end{figure}
	
	
	
	\subsubsection{Functional Annotation}
	Upon performing the \gls{GO} enrichment analysis, a total of 16 \gls{GO} terms were enriched in the ``black'' hippocampus network, 4 in the ``tan'' amygdala network and 45 in the ``yellow'' amygdala network. 
	No \gls{GO} term was enriched in the ``pink'' amygdala network.
	
	The enriched \gls{GO} terms of the ``yellow'' amygdala network were mainly related to translation and transcription and were not specific to brain function or development(\cref{tab:yellowGO}). 
	On the contrary, the \gls{GO} terms enriched in the ``black'' hippocampus network were highly relevant to brain function and development (\cref{tab:blackGO})(e.g. ``central nervous system development'' and ``glutamate metabolic process'') and the ``tan'' amygdala network were also related to ammonium ion metabolism (\cref{tab:tanGO}) which is vita for glutamine synthesis from glutamate\citep{Liaw1995}. 
	
	Together, it is highly likely that the ``black'' hippocampus and ``tan'' amygdala networks are related to brain development and function.
	
	\begin{table}[h]
		\centering
		\caption[\glsentryshort{GO} enrichment results for the ``black'' network from Hippocampus]{\gls{GO} enrichment results for the ``black'' network from Hippocampus.
			Among the enriched \gls{GO} terms, it was most interesting to identify a number of brain developmental related \gls{GO} terms such as ``central nervous system development'', ``axon ensheathment in central nervous system'', ``glutamate metabolic process'' and ``positive regulation of gliogenesis''. 
			Surprisingly, \gls{GO} related to immune systems were also observed ``positive regulation of production of molecular mediator of immune response''.
		}
		\begin{tabular}{rrr}
			\toprule
			term\_ID & description & p-value \\
			\midrule
			GO:0019752 & carboxylic acid metabolic process & $4.92\times 10^{-6}$ \\
			GO:0007417 & central nervous system development & $5.94\times 10^{-5}$ \\
			GO:0002821 & positive regulation of adaptive immune response & $6.12\times 10^{-5}$ \\
			GO:0006082 & organic acid metabolic process & $1.03\times 10^{-3}$ \\
			GO:0032291 & axon ensheathment in central nervous system & $1.86\times 10^{-3}$ \\
			GO:1901565 & organonitrogen compound catabolic process & $1.99\times 10^{-3}$ \\
			GO:0006536 & glutamate metabolic process & $3.54\times 10^{-3}$ \\
			GO:0021762 & substantia nigra development & $3.73\times 10^{-3}$ \\
			GO:0044281 & small molecule metabolic process & $4.34\times 10^{-3}$ \\
			GO:0030194 & positive regulation of blood coagulation & $4.59\times 10^{-3}$ \\
			GO:0009607 & response to biotic stimulus & $6.14\times 10^{-3}$ \\
			GO:0002702 & positive regulation of production of molecular mediator of immune response & $6.21\times 10^{-3}$ \\
			GO:0034103 & regulation of tissue remodeling & $6.21\times 10^{-3}$ \\
			GO:0014015 & positive regulation of gliogenesis & $7.47\times 10^{-3}$ \\
			GO:0098542 & defense response to other organism & $7.95\times 10^{-3}$ \\
			GO:0019835 & cytolysis & $8.72\times 10^{-3}$ \\
			\bottomrule
		\end{tabular}%
		\label{tab:blackGO}%
	\end{table}%
	\begin{table}[h]
		\centering
		\caption[\glsentryshort{GO} enrichment results for the ``tan'' network from Amygdala]{\gls{GO} enrichment results for the ``tan'' network from Amygdala.
			Unlike the ``black'' network, only a small number of \gls{GO} terms were enriched. 
			However, these \gls{GO} terms are relatively specific to amine/ammonium ion metabolism.
			Interestingly, ammonium ion are essential to the synthesis of glutamine from glutamate, suggesting that this network might be relate to the glutamate system.
			}
		\begin{tabular}{rrr}
				\toprule
				term\_ID & description & p-value \\
				\midrule
				GO:0097164 & ammonium ion metabolic process & $1.37\times 10^{-3}$ \\
				GO:0044106 & cellular amine metabolic process & $4.2\times 10^{-3}$ \\
				GO:0009308 & amine metabolic process & $5.41\times 10^{-3}$ \\
				GO:0046519 & sphingoid metabolic process & $6.01\times 10^{-3}$ \\
				\bottomrule
		\end{tabular}%
		\label{tab:tanGO}%
	\end{table}%

	\subsubsection{Associate Co-expression network with \glsentryshort{pgc} schizophrenia data}
	Although the co-expression network were extremely interesting for their expression pattern and functional enrichment in brain development and function related \gls{GO} terms, there were no evidence of their involvement nor importance in schizophrenia.
	Therefore it is of particular interest for us to test whether if genes within these co-expression networks were associated withs schizophrenia. 
	
	First, gene base p-value of 18,622 genes were calculated using p-values from the \gls{pgc} schizophrenia working group\citep{Ripke2014}.
	Gene set enrichment analysis were then performed using \gls{MAGMA}\citep{DeLeeuw2015} to test whether if there genes within the ``black'' hippocampus and ``tan'' amygdala networks were significantly associated with schizophrenia.
	
	Based on the self-contained gene set enrichment analysis, genes within both networks were significantly associated with schizophrenia with p-value of $1.38\times 10^{-41}$ for the ``tan'' amygdala network and $2.70\times 10{-74}$ for the ``black'' hippocampus network.
	These suggest that these networks might be disrupted in schizophrenia patients.
	%Network	Size	self-contained	competitive
	%Amygdala        289   1.3869e-41      0.44715
	%Hippocampus     458   2.6993e-74      0.20002
	
	
	
	
	\subsubsection{Partitioning of Heritability}
	
	\section{Discussion}
	\chapter{Heritability of Response to antipsychotic treatment}
	\chaptermark{Response to antipsychotic treatment}
	%Maybe a section instead of a chapter?
	Important to schizophrenia research
	
	\section{Introduction}
	Here we try to use Beatrice's data and estimate the heritability explained in drug response.
	Should also repeat the region-wise heritability
	\section{Methodology}
	\section{Result}
	\section{Discussion}

	\chapter{Conclusion}
	
	
	
	
	
	
	
	
	\backmatter
	\printbibliography[heading=bibintoc,title={Bibliography}]
	\chapter*{Supplementary Materials}
	\beginsupplement
	\chapter*{Appendix}

\end{document}


