\documentclass[12pt]{book}
\usepackage{amsmath}
\usepackage{bm}
\usepackage{colortbl}
\usepackage{graphicx}
\usepackage{caption}
\usepackage{fullpage}
\usepackage{afterpage}
\usepackage{float}
\usepackage{multirow}
\usepackage[nodisplayskipstretch]{setspace}
\usepackage{booktabs}
\usepackage{textcomp}
\usepackage{gensymb}
\usepackage[utf8]{inputenc} 
%\usepackage{parskip}
\usepackage{xr}
\usepackage{pdflscape}
\usepackage[labelfont=bf,tableposition=top]{caption}
\usepackage[inline]{enumitem}
\usepackage{wrapfig}
\usepackage{longtable}
\usepackage{subfig}
\usepackage[hidelinks]{hyperref}
\usepackage{geometry}
\usepackage{fancyhdr}
\usepackage[acronym,nomain,toc,makeindex ]{glossaries}
\usepackage{graphicx}
\usepackage[section]{placeins}
\usepackage{cleveref} %This need to be the last package for it to work properly 
\usepackage[hyperref=true,maxcitenames=3,maxbibnames=3,url=true,doi=false,isbn=false,backref=true,natbib=true,backend=biber,sorting=nyvt,defernumbers=false, style=authoryear]{biblatex}


%
%
%	TITLE PAGE DEFINITION
%
%
\renewcommand*\contentsname{Contents}

\newcommand{\rom}[1]{\uppercase\expandafter{\romannumeral #1\relax}}
\newcommand*{\glng}{\glsentrylong}
\newcommand*{\Glng}{\Glsentrylong}
\newcommand{\beginsupplement}{%
	\setcounter{table}{0}
	\renewcommand{\thetable}{S\arabic{table}}%
	\setcounter{figure}{0}
	\renewcommand{\thefigure}{S\arabic{figure}}%
}
\newcommand{\specialcell}[2][c]{\begin{tabular}[#1]{@{}c@{}}#2\end{tabular}}

%
%	GLOSSARY SECTION
%
\addbibresource[datatype=bibtex]{citation/citation.bib}
\makeglossary
\newacronym{scz}{SCZ}{schizophrenia}
\newacronym[longplural={Genome Wide Association Studies}]{GWAS}{GWAS}{Genome Wide Association Study}
\newacronym{wgs}{WGS}{Whole Genome Sequencing}
\newacronym{SNP}{SNP}{Single Nucleotide Polymorphism}
\newacronym{LD}{LD}{Linkage Disequilibrium}
\newacronym{PGS}{PGS}{Polygenic Risk Score}
\newacronym{tSVD}{tSVD}{Truncated Singular Value Decomposition}
\newacronym{SVD}{SVD}{Singular Value Decomposition}
\newacronym{shrek}{SHREK}{SNP Heritability and Risk Estimation Kit}
\newacronym{gcta}{GCTA}{Genome-wide Complex Trait Analysis}
\newacronym{ldsc}{LDSC}{LD SCore regression}
\newacronym{CEU}{CEU}{Northern Europeans from Utah}
\newacronym{se}{SE}{standard error}
\newacronym{maf}{maf}{Minor Allele Frequency}
\newacronym{isc}{ISC}{International Schizophrenia Consortium}
\newacronym{mgs}{MGS}{Molecular Genetic of Schizophrenia}
\newacronym{sgene}{SGENE}{Schizophrenia Genetics Consortium}
\newacronym{pgc}{PGC}{Psychiatric Genomics Consortium}
\newacronym{rin}{RIN}{RNA integrity number}
\newacronym{gd}{GD}{Gestation Day}
\newacronym{rpkm}{RPKM}{Reads Per Kilobase per Million mapped reads}
\newacronym{wgcna}{WGCNA}{Weighted Gene Co-expression Network Analysis}
\newacronym{pc}{PC}{Principle Component}
\newacronym{GO}{GO}{Gene Ontology}
\newacronym{MAGMA}{MAGMA}{Multi-marker Analysis of GenoMic Annotation}
\newacronym{ngs}{NGS}{next generation sequencing}
\newacronym{dsm}{DSM}{Diagnostic and Statistical Manual of Mental Disorders}
\newacronym{mse}{MSE}{mean squared error}
\newacronym{who}{WHO}{World Health Organization}
\newacronym{yld}{YLD}{years lost due to disability}
\newacronym{ci}{CI}{confidence interval}
\newacronym{iq}{IQ}{intelligence quotient}
\newacronym{polyic}{PolyI:C}{polyriboinosinic-polyribocytidilic acid}
\newacronym{lps}{LPS}{lipopolysaccharide}
\newacronym{il6}{IL-6}{Interleukin-6}
\newacronym{mz}{MZ}{monozygotic}
\newacronym{dz}{DZ}{dizygotic}
\newacronym{mia}{MIA}{maternal immune activation}
\newacronym{ncp}{NCP}{non-centrality parameter}
\newacronym{cnv}{CNV}{copy number variation}
\newacronym{grm}{GRM}{Genetic Relationship Matrix}
\newacronym{gc}{GC}{Genomic Control}
\newacronym{mhc}{MHC}{major histocompatibility complex}
\newacronym{cns}{CNS}{central nervous system}
\newacronym{panss}{PANSS}{Positive and Negative Symptom Scale}
\newacronym{fga}{FGA}{First Generation Antipsychotic}
\newacronym{sga}{SGA}{Second Generation Antipsychotic}
\newacronym{nmda}{NMDA}{N-methyl-D-aspartate}
\newacronym{eps}{EPS}{extrapyramidal motor symptoms}
\newacronym{td}{TD}{Tardive dyskinesia}
\newacronym{pet}{PET}{positron emission tomography}
\newacronym{catie}{CATIE}{Clinical Antipsychotic Trials of Intervention Effectiveness}
\newacronym{pca}{PCA}{principle component analysis}
\newacronym{fdr}{FDR}{false discovery rate}
\newacronym{pcgc}{PCGC}{Phenotype correlation - genotype correlation regression}
\newacronym{scidp}{SCID-P}{Structured Clinical Interview for DSM-IV-TR Axis I Disorders, Research Version, Patient Edition}
\newacronym{ibd}{IBD}{identity by descent}
\newacronym{bp}{bp}{base pair}
\newacronym{mb}{mb}{megabase}
\newacronym{deg}{DEG}{differentially expressed gene}
\newacronym{qc}{QC}{quality control}
\newacronym{hku}{HKU}{the University of Hong Kong}
\newacronym{pcr}{PCR}{polymerase chain reaction}
\newacronym{ercc}{ERCC}{External RNA Controls Consortium}
\newacronym{go}{GO}{Gene Ontology}
\newacronym{rtpcr}{rt-PCR}{real time PCR}
\newacronym{ct}{CT}{cycle threshold}
\newacronym{geo}{GEO}{Gene Expression Omnibus}
\newacronym{mlm}{MLM}{mixed linear model}
\newacronym{reml}{REML}{restricted maximum likelihood}
\newacronym{pufa}{PUFA}{polyunsaturated fatty acid}
\newacronym{lrt}{LRT}{likelihood ratio test}
\newacronym{msigdb}{MSigDB}{Molecular Signatures Database}
\newacronym{fgf}{FGF}{fibroblast growth factor}
\newacronym{egf}{EGF}{epidermal growth factor}
\newacronym{pi3k}{PI3K}{phosphatidylinositol 3-kinase}
\newacronym{mapk}{MAPK}{mitogen-activated protein kinase}
\newacronym{gaba}{GABA}{$\gamma$-aminobutyric acid}
%
%
%	FORMATING SECTION
%
%

\pagestyle{fancy}
\fancyhf{}
\fancyfoot[LE,RO]{\thepage}
\renewcommand{\footrulewidth}{1pt}
\fancyhead[LE]{\leftmark}
\fancyhead[RO]{\rightmark}

\geometry{
	top=1in,            % <-- you want to adjust this
	inner=1in,
	outer=3.5cm,
	bottom=3.5cm,
	headheight=3ex,       % <-- and this
	headsep=2ex,          % <-- and this
}


\raggedbottom %Remove it before printing as this is something to do with global settings. Can make each page look uneven but more dense. 
%\onehalfspacing
\doublespacing
%TODO this is the include space
%\includeonly{heritability_estimation}
\includeonly{omega_project}
%\includeonly{literature_review}


%\extrafloats{100}






\makeindex
\begin{document}\thispagestyle{empty}
\pagestyle{empty}

%\maketitle
\begin{singlespace}
	\begin{titlepage}
		\begin{center}
			\vspace*{1cm}
			
			\Huge
			\textbf{Genetic and Environmental risk factors of Schizophrenia}
			
			\vspace{0.5cm}
			\LARGE
			
			\vspace{1.5cm}
			
			\textbf{\href{mailto:choishingwan@gmail.com}{Choi Shing Wan}}
			
			\vfill
			
			A thesis submitted in partial fulfillment of the requirements for \\
			the Degree of Doctor of Philosophy
			
			\vspace{0.8cm}
			
			\includegraphics[width=0.4\textwidth]{figure/hkuLogo.jpg}
			
			\Large
			Department of Psychiatry\\
			University of Hong Kong\\
			Hong Kong\\
			\today
			
		\end{center}
	\end{titlepage}
\end{singlespace}


\frontmatter 
	\cleardoublepage
	\phantomsection
	\addcontentsline{toc}{chapter}{Abstract}
	\chapter*{\centerline{Abstract}}
	Schizophrenia is a complex disorder affecting approximately 1\% of the population worldwide. 
	Twin studies and population studies suggest that genetic variations accounts for $\sim80\%$ of the population variance in disease liability of schizophrenia.
	Yet the true contribution of common genetic variants, particularly \gls{SNP}, to schizophrenia remains unknown.
	
	
	
	In this thesis, we proposed an alternative approach for estimating \gls{SNP}-heritability from \gls{GWAS} summary statistics, called the \gls{shrek}, which provides a robust heritability estimation, even under oligogenic conditions.
	Our simulation results suggest that when compared to \gls{ldsc}, \gls{shrek} can provide a more robust estimate for oligogentic traits and in case-control designs where no confounding variables was present.
	Using the latest \gls{GWAS} summary statistics released by the Psychiatric Genomic Consortium , we estimated that schizophrenia has a \gls{SNP}-heritability of 0.174 (SD=0.00453), which is similar to the estimate of 0.197 (SD=0.0058) by \gls{ldsc}.
	
	Finally, 
	
	
	
	With the success of the Psychiatric Genomic Consortium Schizophrenia \gls{GWAS}, it is of immense interest to accurately determine the true contribution of common genetic variiants, particularly \gls{SNP} (\gls{SNP}-heritability), to schizophrenia.
	Combined with the heritability estimation from twin studies, this should allow one to determine whether if majority of the heritability has been captured through the \gls{GWAS} study.
	A discrepancy between the twin heritability and the \gls{SNP}-heritability might suggest that some of the genetic variations were inherited through other mechanism such as epigenetics changes, structural variations and even gene environmental interactions.
	Thus, the \gls{SNP} heritability will provide valuable information that may guide future researches for schizophrenia.
	
	The problem with \gls{SNP} heritability was that most algorithms require the use of raw genotypes from the \gls{GWAS} in order to estimate the \gls{SNP} heritability.
	However, due to privacy concerns, the raw genotypes from large scale \gls{GWAS} such as that for schizophrenia were usually unavailable. 
	This leads to difficulties in the estimation of \gls{SNP} heritability.
	Recently, the \gls{ldsc} has been published which has the ability to estimates the \gls{SNP} heritability with only the summary statistics from the \gls{GWAS}, thus does not require the use of raw genotype.
	However, \gls{ldsc} is ill suited for estimation of heritability under oligogenic conditions because it requires regressing genetic effect as function of linkage disequilibrium over large number of \glspl{SNP}.
	Although schizophrenia are most likely polygenic, it is still of particular interest to see if there can be robust algorithms that may accurately estimate the \gls{SNP}-heritability of schizophrenia.
	
	The ultimate goal of schizophrenia research is usually to be able to develop possible treatment or cure for schizophrenia patients, thus improving their quality of life. 
	
	\glsresetall
	\cleardoublepage
	\phantomsection
	\addcontentsline{toc}{chapter}{Declaration}
	\newgeometry{inner=2in, outer=2in}
	\chapter*{\centerline{Declaration}}
	\vspace{1cm}
	I declare that this thesis represents my own work, except where due acknowledgments	is made, and that it has not been previously included in a thesis, dissertation or report submitted to this University or to any other institution for a degree, diploma or other qualification.
	\vspace{1.5cm}
	
	\centerline{Signed....................................................................}
	\restoregeometry
	\cleardoublepage
	\phantomsection
	\addcontentsline{toc}{chapter}{Acknowledgments}
	\chapter*{\centerline{Acknowledgements}}
	\cleardoublepage
	\phantomsection
	\begin{singlespace}
	\printglossary[title=Abbreviations,toctitle=Abbreviations]
	\cleardoublepage
	\phantomsection
	%To generate the correct abbreviations, use alt+shift+F1 twice before using F1
	
	\cleardoublepage
	\phantomsection
	\addcontentsline{toc}{chapter}{Contents}
		\tableofcontents
		\listoffigures
		\listoftables
	\end{singlespace}
\mainmatter
\pagestyle{fancy}

\setlength{\parindent}{4em}
\setlength{\parskip}{0.75em}
	
	\include{literature_review}
	\include{heritability_estimation}
	\include{omega_project}
	\chapter{Conclusion}
	\gls{shrek}, an algorithm for the estimation of heritability using \gls{GWAS} test statistics are reported in this thesis.
	To our knowledge, this is the only algorithm other than the LD Score regression that can perform heritability estimation using test statistics.
	In this thesis, we were able to demonstrate that \gls{shrek} can provided a more robust estimate in case-control designs when no confounding variables was present. 
	By applying \gls{shrek} on the test statistic from the \gls{pgc} \glng{scz} \gls{GWAS}, we estimated that \glng{scz} has a \gls{SNP}-heritability of 0.174 (SD=0.00453), which is similar to the estimate of 0.197 (SD=0.0058) by \gls{ldsc}.
	
	On the other hand, we report a pilot RNA Sequencing study aiming to investigate the effect of \acrfull{mia} and n-3 \gls{pufa} rich diet on the gene expression pattern in the adult cerebellum. 
	Overall, our results suggest the \gls{mia} exposure might disrupt gene expressions that are related to neural function in the cerebellum of adult mice. 
	In addition, we observed there to be a significant up-regulation of \textit{Sgk1} after the \gls{polyic} exposed mice given the n-3 \gls{pufa} rich diet. 
	Consider the neuroprotective nature of \textit{Sgk1}, it is possible that the n-3 \gls{pufa} rich diet can help to ``rescued'' some of the neuronal function by up-regulating \textit{Skg1}.
	As a pilot study, our results also helps to design a follow up study with adequate sample size and control for different confounding variables such as batch effect and cage effect. 
	
	\section{Challenge in SNP-Heritability Estimation}
	Although now that we can estimates the \gls{SNP} heritability based on the test statistic of large scale meta analysis, this is only the beginning for there are still a lot of questions left unanswered in the estimation of \gls{SNP} heritability.
	One major problem of \gls{shrek} and \gls{ldsc} is that they both heavily relies on the \gls{LD} structures from the reference panel.
	However, for any meta analysis, samples are usually from a large variety of the ethnicity, although one can choose a reference that are representative of the majority of samples, we are uncertain how the sample mixing can alter the \gls{LD} structures, this might potentially leads to biased estimates.
	If the fundamental \gls{LD} structure was not as expected, both \gls{shrek} and \gls{ldsc} will not be able to provide an accurate estimate. 
	For example, if a \gls{GWAS} was conducted with 50\% European and 50\% African, population stratification may confound the results.
	Even if one control for the population stratification using the \gls{pca}, the question remains whether if one should use the African reference panel or the European reference panel in the estimation of \gls{SNP} heritability.
	This problem is further complicated when the information regarding population stratification was usually unavailable. 
	Thus further researches are required to tackle the problem of population stratification before one can confidently estimate the \gls{SNP} heritability on large scale meta analysis that consists of samples from a large variety of ethic background.
	
	One important observation in our simulation study was that there was a general bias observed in all the \gls{SNP}-heritability estimation algorithm.
	This is likely to due to the ascertainment bias introduced through case control sampling.
	Although the liability adjustment was performed, bias was still observed. 
	This suggested that we will need a better liability adjustment algorithm if we would like to accurately estimate the \gls{SNP}-heritability from case control studies.
	
	As technology advances, researchers can now use the \gls{ngs} technology to sequence the genome at per base resolution. 
	This brings great prospect in the genetic studies for now we can directly identify the causal variants and can even detect rare causal variants providing sufficient sample size. 
	However, both \gls{shrek} and \gls{ldsc} are designed to work on the test statistics of a \gls{GWAS} where common \glspl{SNP} are usually the focus of the studies. 
	Because of the huge sampling error associating with rare variants, \gls{shrek} and \gls{ldsc} might be unsuitable for rare variants. 
	In fact, it was found that when all causal variants are rare (\gls{maf} $<1\%$), \gls{ldsc} will often generate a negative slope, and the intercept will exceed the mean $\chi^2$ statistic \citep{Bulik-Sullivan2015}.
	As a result of that, a different algorithm must be developed in order to estimates the heritability from rare variants.
	
	\section{Schizophrenia: Future Perspectives}
	With the success of the \gls{pgc} \glng{scz} \gls{GWAS}, research in \glng{scz} genetics has finally entered an era of success.
	Through international collaboration we have finally identify 108 genetic loci that are associated with \glng{scz} \citep{Ripke2013}.
	However, the \gls{GWAS} only provides statistical association between the variants \glng{scz} and does not provide direct evidence as to the functional involvement of these variants in the etiology of \glng{scz}.
	Functional consequences of these variations in \glng{scz} are therefore an important topic for the understanding of the mechanism of \glng{scz}.
	
	On the other hand, when estimating the \gls{SNP}-heritability of \glng{scz}, it was found that no more than 20\% of the heritability has been accounted for by the current \gls{GWAS}.
	There is no doubt that by continue to increase the sample size of the \gls{GWAS}, one can identify more variants associated with \glng{scz} and therefore increases the \gls{SNP}-heritability. 
	However, it is also likely that they will have a very small effect size which might not be useful for clinical translations.
	Another possibility is that the so call ``missing'' heritability might be accounted for by other factors such as rare variants and epigenetic such as methylation.
	
	There is clear evidence that \glng{scz} patients has a higher mortality than the general population \citep{Saha2007}.
	Given this strong selective pressure, it is likely that the causal variants of \glng{scz} that has a large effect size will be selected against in the population. 
	As a result of that, causal variants with large effect size should be rare in nature (\cref{fig:effectSize}).
	With the technological advancement in \gls{ngs}, we are now able to investigate the human genome at per base resolution using Exome Sequencing and even Whole Genome Sequencing technology.
	Recent study by \citet{Purcell2014} was able to identify gene sets enriched by rare variants that were associated with \glng{scz} using Exome Sequencing. 
	This demonstrate the power of the sequencing technology in the identification of possible risk variants. 
	Moreover, there was overlaps observed between genes harboring rare risk variants and those within the \gls{pgc} \glng{scz} \gls{GWAS} \citep{Purcell2014}, suggesting that the rare variants and common variants studies are complementing each other.
	As more resources are devoted in to sequencing the genome of \glng{scz} patients, we would expect to identify more rare variants that are associated with \glng{scz}.
	\begin{figure}
		\centering
		\includegraphics[width=0.7\textwidth]{figure/maf_effectSize.png}
		\caption[Relationship between Effect Size and Allele Frequency]{
			Relationship between effect size and allele frequency. 
			It is expected that rare variants with large effect size were actively selected against in the population and therefore should be rare.
		}
		\label{fig:effectSize}
	\end{figure}
	
	Currently, most of the focus in \glng{scz} was directed to genetic variation yet it is possible that the heritability of \glng{scz} is also transmitted in the form of epigenetic changes such as methylation.
	When considering the risk of \glng{scz}, it was observed that the risk for individual born from a schizophrenic mother is larger than that from a schizophrenic father. 
	It is therefore possible that the risk of \glng{scz} is passed on through maternal imprinting. 
	Epigenetic studies in \glng{scz} \citep{Wockner2014,Nishioka2012} has identified genes with differential DNA methylation patterns associated with \glng{scz}, suggesting the important of epigenetic in the etiology of \glng{scz}.
	
	As a genetic disorder, most of the research of \glng{scz} has been focusing on the genetic factors. 
	Although the genetic variation accounted for majority of the variations in \glng{scz}, the environmental factors, especially prenatal infection are also important factors to consider. 
	It was estimated that prenatal infection accounts for roughly 33\% of all \glng{scz} cases \citep{Brown2010}.
	The \gls{mia} rodent model has provide vital information on the possible interaction between the immune and neuronal system in the etiology of \glng{scz} \citep{Meyer2007a}.
	For example, \gls{il6}, a pro-inflammatory cytokine has been found to be an important mediator in generating the schizophrenia-like behaviour in rodent model \citep{Smith2007}.
	More importantly, there are evidence of the interaction between prenatal infection and genetic variation, supporting a mechanism of gene-environment interaction in the causation of \glng{scz} \citep{Clarke2009}.
	As the \gls{SNP}-heritability estimation does not take into account of the gene environmental interactions, it is possible that the ``missing'' heritability can be due to gene-environmental interactions. 
	Efforts is now made by the European network of national \glng{scz} networks studying Gene-Environmental Interaction (EUGEI) to identify possible genetic and environmental interaction that contributes to the disease etiology of \glng{scz}.
	
	
	% We know too little to understand
	With the sophistication of technologies, we can now perform whole genome sequencing with the HiSeq \RN{10} Ten system costing less than \$1,000. 
	The problem now then isn't the cost of generating the data, but the difficulties in making sense of the data. 
	The first problem is the alignment of sequence read to low complexity sequence or low-degeneracy repeats \citep{Sims2014}. 
	One possible solution is to use system such as the Oxford Nanopore which can provide extra long-reads, thus allowing for better alignment.
	However, the Oxford Nanopore is still underdevelopment and has a relatively high error rate \citep{Mikheyev2014}. 
	Only until the error rate is dramatically decreased can the use of Oxford Nanopore system become feasible. 
	
	Even if we can accurately align all the reads to the genome, we still face another challenge.
	When it comes to complex disease such as \glng{scz}, there can be a lot of causal variants observed throughout the genome. 
	However, we are only capable to estimate the functional impact of variants on the exomic regions.
	The development of ENCODE project \citep{ENCODEProjectConsortium2012} and Genotype-Tissue Expression (GTEx) project \citep{Consortium2015} have helped provide reference point for the annotation of genetic variations in the intergenic regions yet there are still many genetic variation in the genome where their function remains unknown to us. 
	Only through the tireless effort of the molecular biologist can we gain sufficient information required to make sense of the sequencing data obtained.	
	
	% This is only the beginning, not the end
	In conclusion, we have only catch a glimpse of the etiology of \glng{scz} and there are still a lot of questions left unanswered.
	It is expected that only by combining the study of epigenetic, genomic variation, gene expressions, and gene environmental interaction can we have a deeper understanding of the complex disease mechanism of \glng{scz}.
	Hopefully, in the near future, we can gain enough understanding to start translating the research findings into clinical applications to help improving the quality of life of \glng{scz} patients.
	
	
	\backmatter
	\printbibliography[heading=bibintoc,title={Bibliography}]

	%\input{environmental_risk/er_rtPCR_primer.tex}
\end{document}


